<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.4.554">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="clee">

<title>MD &amp; CH - Repurposing desipramine for neurological disorders?</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../img/mi2.PNG" rel="icon" type="image/png">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<style>html{ scroll-behavior: smooth; }</style>
<link href="../site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">

<script src="../site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>

<link href="../site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">

<script src="../site_libs/datatables-binding-0.33/datatables.js"></script>

<script src="../site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>

<link href="../site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet">

<link href="../site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet">

<script src="../site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>

<link href="../site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">

<script src="../site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>



<link rel="stylesheet" href="../styles/custom.scss">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a href="../index.html" class="navbar-brand navbar-brand-logo">
    <img src="../img/mi_logo.png" alt="" class="navbar-logo">
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../gallery.html"> 
<span class="menu-text">Blog</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../about.html"> 
<span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../contact.html"> 
<span class="menu-text">Contact</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
          <div class="quarto-navbar-tools tools-wide">
    <a href="https://youtube.com" title="" class="quarto-navigation-tool px-1" aria-label=""><i class="bi bi-youtube"></i></a>
    <div class="dropdown">
      <a href="" title="" id="quarto-navigation-tool-dropdown-0" class="quarto-navigation-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false" aria-label=""><i class="bi bi-github"></i></a>
      <ul class="dropdown-menu" aria-labelledby="quarto-navigation-tool-dropdown-0">
          <li>
            <a class="dropdown-item quarto-navbar-tools-item" href="https://code.com">
            Source Code
            </a>
          </li>
      </ul>
    </div>
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#section" id="toc-section" class="nav-link active" data-scroll-target="#section"></a></li>
  <li><a href="#overview" id="toc-overview" class="nav-link" data-scroll-target="#overview">Overview</a>
  <ul class="collapse">
  <li><a href="#row" id="toc-row" class="nav-link" data-scroll-target="#row">Row</a>
  <ul class="collapse">
  <li><a href="#figure-1" id="toc-figure-1" class="nav-link" data-scroll-target="#figure-1">Figure 1</a></li>
  <li><a href="#column" id="toc-column" class="nav-link" data-scroll-target="#column">Column</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results">Results</a>
  <ul class="collapse">
  <li><a href="#row-1" id="toc-row-1" class="nav-link" data-scroll-target="#row-1">Row</a></li>
  <li><a href="#row-2" id="toc-row-2" class="nav-link" data-scroll-target="#row-2">Row</a>
  <ul class="collapse">
  <li><a href="#column-1" id="toc-column-1" class="nav-link" data-scroll-target="#column-1">Column</a></li>
  <li><a href="#column-2" id="toc-column-2" class="nav-link" data-scroll-target="#column-2">Column</a></li>
  </ul></li>
  <li><a href="#row-3" id="toc-row-3" class="nav-link" data-scroll-target="#row-3">Row</a></li>
  </ul></li>
  <li><a href="#data" id="toc-data" class="nav-link" data-scroll-target="#data">Data</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Repurposing desipramine for neurological disorders?</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>clee </p>
          </div>
  </div>
    
  
    
  </div>
  


</header>


<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co">#rm(list=ls())</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(reshape)</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(scales)</span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(DT)</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(gt)</span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(data.table)</span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggVennDiagram)</span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a><span class="fu">theme_set</span>(<span class="fu">theme_minimal</span>(<span class="at">base_size =</span> <span class="dv">24</span>, <span class="at">base_family =</span> <span class="st">"Atkinson Hyperlegible"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a>dt <span class="ot">&lt;-</span> <span class="fu">read_csv</span>(<span class="st">"C:/Users/quicl/Downloads/asm.csv"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a>dt <span class="ot">&lt;-</span> dt <span class="sc">|&gt;</span></span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span><span class="fu">is.na</span>(name))<span class="sc">|&gt;</span></span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(</span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a>    <span class="co">#category = str_to_sentence(category),</span></span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a>    <span class="co">#Name = str_to_sentence(Name),</span></span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">prot_func =</span> <span class="fu">paste</span>(name,func1,func2,func3,func4,func5)</span>
<span id="cb3-7"><a href="#cb3-7" aria-hidden="true" tabindex="-1"></a>    )<span class="sc">|&gt;</span></span>
<span id="cb3-8"><a href="#cb3-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(check,name, change, change_desi, change_mango,prot_func)<span class="sc">|&gt;</span></span>
<span id="cb3-9"><a href="#cb3-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">vesicle =</span> <span class="fu">str_detect</span>(prot_func, <span class="st">"esicl"</span>, <span class="at">negate =</span> <span class="cn">FALSE</span>),</span>
<span id="cb3-10"><a href="#cb3-10" aria-hidden="true" tabindex="-1"></a>         <span class="at">kinase=</span> <span class="fu">str_detect</span>(prot_func, <span class="st">"ase"</span>, <span class="at">negate =</span> <span class="cn">FALSE</span>),</span>
<span id="cb3-11"><a href="#cb3-11" aria-hidden="true" tabindex="-1"></a>         <span class="at">autophagy =</span> <span class="fu">str_detect</span>(prot_func, <span class="st">"utophagy"</span>, <span class="at">negate =</span> <span class="cn">FALSE</span>),</span>
<span id="cb3-12"><a href="#cb3-12" aria-hidden="true" tabindex="-1"></a>         <span class="at">apoptosis =</span> <span class="fu">str_detect</span>(prot_func, <span class="st">"poptosi"</span>, <span class="at">negate =</span> <span class="cn">FALSE</span>))<span class="sc">|&gt;</span></span>
<span id="cb3-13"><a href="#cb3-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">prot_function =</span> <span class="fu">case_when</span>(</span>
<span id="cb3-14"><a href="#cb3-14" aria-hidden="true" tabindex="-1"></a>        autophagy <span class="sc">==</span> <span class="cn">TRUE</span>   <span class="sc">~</span> <span class="st">"autophagy"</span>,</span>
<span id="cb3-15"><a href="#cb3-15" aria-hidden="true" tabindex="-1"></a>        apoptosis <span class="sc">==</span> <span class="cn">TRUE</span>   <span class="sc">~</span> <span class="st">"apoptosis"</span>,</span>
<span id="cb3-16"><a href="#cb3-16" aria-hidden="true" tabindex="-1"></a>        vesicle <span class="sc">==</span> <span class="cn">TRUE</span>  <span class="sc">~</span> <span class="st">"vesicle trafficking"</span>,</span>
<span id="cb3-17"><a href="#cb3-17" aria-hidden="true" tabindex="-1"></a>        kinase <span class="sc">==</span> <span class="cn">TRUE</span>   <span class="sc">~</span> <span class="st">"kinase &amp; enzyme related"</span>,</span>
<span id="cb3-18"><a href="#cb3-18" aria-hidden="true" tabindex="-1"></a>      <span class="at">.default =</span> <span class="st">"others"</span>))<span class="sc">|&gt;</span>  </span>
<span id="cb3-19"><a href="#cb3-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">siRNA_change =</span> <span class="fu">case_when</span>(</span>
<span id="cb3-20"><a href="#cb3-20" aria-hidden="true" tabindex="-1"></a>      change <span class="sc">&gt;</span> <span class="dv">2</span> <span class="sc">~</span> <span class="st">"Decrease"</span>,</span>
<span id="cb3-21"><a href="#cb3-21" aria-hidden="true" tabindex="-1"></a>      change <span class="sc">&lt;</span> <span class="fl">0.5</span> <span class="sc">~</span> <span class="st">"Increase"</span>,</span>
<span id="cb3-22"><a href="#cb3-22" aria-hidden="true" tabindex="-1"></a>      <span class="at">.default =</span> <span class="st">"No change"</span>))<span class="sc">|&gt;</span></span>
<span id="cb3-23"><a href="#cb3-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">desi_change =</span> <span class="fu">case_when</span>(</span>
<span id="cb3-24"><a href="#cb3-24" aria-hidden="true" tabindex="-1"></a>      change_desi <span class="sc">&gt;</span> <span class="dv">2</span> <span class="sc">~</span> <span class="st">"Decrease"</span>,</span>
<span id="cb3-25"><a href="#cb3-25" aria-hidden="true" tabindex="-1"></a>      change_desi <span class="sc">&lt;</span> <span class="fl">0.5</span> <span class="sc">~</span> <span class="st">"Increase"</span>,</span>
<span id="cb3-26"><a href="#cb3-26" aria-hidden="true" tabindex="-1"></a>      <span class="at">.default =</span> <span class="st">"No change"</span>))<span class="sc">|&gt;</span></span>
<span id="cb3-27"><a href="#cb3-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">mango_change =</span> <span class="fu">case_when</span>(</span>
<span id="cb3-28"><a href="#cb3-28" aria-hidden="true" tabindex="-1"></a>      change_mango <span class="sc">&gt;</span> <span class="dv">2</span> <span class="sc">~</span> <span class="st">"Decrease"</span>,</span>
<span id="cb3-29"><a href="#cb3-29" aria-hidden="true" tabindex="-1"></a>      change_mango <span class="sc">&lt;</span> <span class="fl">0.5</span> <span class="sc">~</span> <span class="st">"Increase"</span>,</span>
<span id="cb3-30"><a href="#cb3-30" aria-hidden="true" tabindex="-1"></a>      <span class="at">.default =</span> <span class="st">"No change"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<section id="section" class="level1 sidebar">
<h1 class="sidebar"></h1>
<p><strong>Results</strong></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-1-contents" aria-controls="callout-1" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Table
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-1" class="callout-1-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<table class="table">
<tbody>
<tr class="odd">
<td><strong>Total Proteins found</strong></td>
<td>870</td>
</tr>
<tr class="even">
<td><strong>ASM Sensitive</strong></td>
<td>227</td>
</tr>
<tr class="odd">
<td><strong>Desipramine Sensitive</strong></td>
<td>206</td>
</tr>
<tr class="even">
<td><strong>Mangostin Sensitive</strong></td>
<td>208</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<hr>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-2-contents" aria-controls="callout-2" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Figure legends
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-2" class="callout-2-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><strong>Fig 1</strong>: Ontology analysis on identified 870 proteins. <strong>Fig 2</strong>: Around 40% of changes overlapped between treatments. <strong>Fig 3</strong>: The amount of protein changes in Desipramine and Mangostin treated samples were similar. More than 60% proteins remain unchanged after ASM reduction. <strong>Fig 4</strong>: Functions of protein changes in different treatments(a-c). <strong>Fig 5</strong>: Top 4 proteins affected after ASM reduction involved in (a) vesicle trafficking (b) apoptosis (c) enzymatic activities.</p>
</div>
</div>
</div>
<hr>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-3-contents" aria-controls="callout-3" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Summary
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-3" class="callout-3-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Our data shows the amount of autophagy related proteins were increased in lipid raft after ASM inhibition through siASM. However, the amount remained the same or decreased in cells treated with Mangostin and Desipramine, respectively.</p>
</div>
</div>
</div>
<hr>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-4-contents" aria-controls="callout-4" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
References
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-4" class="callout-4-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<ol type="1">
<li>Xiong X, Lee CF, Li W, Yu J, Zhu L, Kim Y, Zhang H, Sun H. Acid Sphingomyelinase regulates the localization and trafficking of palmitoylated proteins. Biol Open. 2019 Oct 15;8(10):bio040311. doi: 10.1242/bio.040311. PMID: 31142470; PMCID: PMC6826292.</li>
<li>Petrosyan E, Fares J, Cordero A, Rashidi A, Arrieta VA, Kanojia D, Lesniak MS. Repurposing autophagy regulators in brain tumors. Int J Cancer. 2022 Jul 15;151(2):167-180. doi: 10.1002/ijc.33965. Epub 2022 Mar 4. PMID: 35179776; PMCID: PMC9133056.</li>
</ol>
</div>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>n_db <span class="ot">&lt;-</span> <span class="fu">nrow</span>(dt)</span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>p_vesicle <span class="ot">&lt;-</span> dt <span class="sc">|&gt;</span></span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>(vesicle) <span class="sc">|&gt;</span></span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">p =</span> n <span class="sc">/</span> <span class="fu">sum</span>(n)) <span class="sc">|&gt;</span></span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(vesicle <span class="sc">==</span> <span class="cn">TRUE</span>) <span class="sc">|&gt;</span></span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pull</span>(p)</span>
<span id="cb4-8"><a href="#cb4-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-9"><a href="#cb4-9" aria-hidden="true" tabindex="-1"></a>p_vesicle_color <span class="ot">&lt;-</span> <span class="fu">case_when</span>(</span>
<span id="cb4-10"><a href="#cb4-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">between</span>(p_vesicle, params<span class="sc">$</span>dist_rate,  params<span class="sc">$</span>dist_rate <span class="sc">+</span> params<span class="sc">$</span>threshold_diff) <span class="sc">~</span> <span class="st">"warning"</span>,</span>
<span id="cb4-11"><a href="#cb4-11" aria-hidden="true" tabindex="-1"></a>  p_vesicle <span class="sc">&gt;</span> params<span class="sc">$</span>dist_rate <span class="sc">+</span> params<span class="sc">$</span>threshold_diff <span class="sc">~</span> <span class="st">"danger"</span>,</span>
<span id="cb4-12"><a href="#cb4-12" aria-hidden="true" tabindex="-1"></a>  <span class="at">.default =</span> <span class="st">"light"</span></span>
<span id="cb4-13"><a href="#cb4-13" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb4-14"><a href="#cb4-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-15"><a href="#cb4-15" aria-hidden="true" tabindex="-1"></a>p_autophagy <span class="ot">&lt;-</span> dt <span class="sc">|&gt;</span></span>
<span id="cb4-16"><a href="#cb4-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>(autophagy) <span class="sc">|&gt;</span></span>
<span id="cb4-17"><a href="#cb4-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">p =</span> n <span class="sc">/</span> <span class="fu">sum</span>(n)) <span class="sc">|&gt;</span></span>
<span id="cb4-18"><a href="#cb4-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(autophagy <span class="sc">==</span> <span class="cn">TRUE</span>) <span class="sc">|&gt;</span></span>
<span id="cb4-19"><a href="#cb4-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pull</span>(p)</span>
<span id="cb4-20"><a href="#cb4-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-21"><a href="#cb4-21" aria-hidden="true" tabindex="-1"></a>p_autophagy_color <span class="ot">&lt;-</span> <span class="fu">case_when</span>(</span>
<span id="cb4-22"><a href="#cb4-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">between</span>(p_autophagy,params<span class="sc">$</span>dist_rate,  params<span class="sc">$</span>dist_rate <span class="sc">+</span> params<span class="sc">$</span>threshold_dif) <span class="sc">~</span> <span class="st">"warning"</span>,</span>
<span id="cb4-23"><a href="#cb4-23" aria-hidden="true" tabindex="-1"></a>  p_autophagy <span class="sc">&gt;</span> params<span class="sc">$</span>dist_rate <span class="sc">+</span> params<span class="sc">$</span>threshold_diff <span class="sc">~</span> <span class="st">"danger"</span>,</span>
<span id="cb4-24"><a href="#cb4-24" aria-hidden="true" tabindex="-1"></a>  <span class="at">.default =</span> <span class="st">"light"</span></span>
<span id="cb4-25"><a href="#cb4-25" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb4-26"><a href="#cb4-26" aria-hidden="true" tabindex="-1"></a>p_apoptosis <span class="ot">&lt;-</span> dt <span class="sc">|&gt;</span></span>
<span id="cb4-27"><a href="#cb4-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>(apoptosis) <span class="sc">|&gt;</span></span>
<span id="cb4-28"><a href="#cb4-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">p =</span> n <span class="sc">/</span> <span class="fu">sum</span>(n)) <span class="sc">|&gt;</span></span>
<span id="cb4-29"><a href="#cb4-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(apoptosis <span class="sc">==</span> <span class="cn">TRUE</span>) <span class="sc">|&gt;</span></span>
<span id="cb4-30"><a href="#cb4-30" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pull</span>(p)</span>
<span id="cb4-31"><a href="#cb4-31" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-32"><a href="#cb4-32" aria-hidden="true" tabindex="-1"></a>p_apoptosis_color <span class="ot">&lt;-</span> <span class="fu">case_when</span>(</span>
<span id="cb4-33"><a href="#cb4-33" aria-hidden="true" tabindex="-1"></a>  <span class="fu">between</span>(p_apoptosis,params<span class="sc">$</span>dist_rate,  params<span class="sc">$</span>dist_rate <span class="sc">+</span> params<span class="sc">$</span>threshold_dif) <span class="sc">~</span> <span class="st">"warning"</span>,</span>
<span id="cb4-34"><a href="#cb4-34" aria-hidden="true" tabindex="-1"></a>  p_apoptosis <span class="sc">&gt;</span> params<span class="sc">$</span>dist_rate <span class="sc">+</span> params<span class="sc">$</span>threshold_diff <span class="sc">~</span> <span class="st">"danger"</span>,</span>
<span id="cb4-35"><a href="#cb4-35" aria-hidden="true" tabindex="-1"></a>  <span class="at">.default =</span> <span class="st">"light"</span></span>
<span id="cb4-36"><a href="#cb4-36" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb4-37"><a href="#cb4-37" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-38"><a href="#cb4-38" aria-hidden="true" tabindex="-1"></a>p_siASM_nc <span class="ot">&lt;-</span> dt <span class="sc">|&gt;</span></span>
<span id="cb4-39"><a href="#cb4-39" aria-hidden="true" tabindex="-1"></a>    <span class="fu">count</span>(siRNA_change) <span class="sc">|&gt;</span></span>
<span id="cb4-40"><a href="#cb4-40" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="at">p =</span> n <span class="sc">/</span> <span class="fu">sum</span>(n)) <span class="sc">|&gt;</span></span>
<span id="cb4-41"><a href="#cb4-41" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(siRNA_change <span class="sc">!=</span> <span class="st">"No change"</span>) <span class="sc">|&gt;</span></span>
<span id="cb4-42"><a href="#cb4-42" aria-hidden="true" tabindex="-1"></a>    <span class="fu">pull</span>(p)</span>
<span id="cb4-43"><a href="#cb4-43" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-44"><a href="#cb4-44" aria-hidden="true" tabindex="-1"></a>p_desi_nc <span class="ot">&lt;-</span> dt <span class="sc">|&gt;</span></span>
<span id="cb4-45"><a href="#cb4-45" aria-hidden="true" tabindex="-1"></a>    <span class="fu">count</span>(desi_change) <span class="sc">|&gt;</span></span>
<span id="cb4-46"><a href="#cb4-46" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="at">p =</span> n <span class="sc">/</span> <span class="fu">sum</span>(n)) <span class="sc">|&gt;</span></span>
<span id="cb4-47"><a href="#cb4-47" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(desi_change <span class="sc">!=</span> <span class="st">"No change"</span>) <span class="sc">|&gt;</span></span>
<span id="cb4-48"><a href="#cb4-48" aria-hidden="true" tabindex="-1"></a>    <span class="fu">pull</span>(p)</span>
<span id="cb4-49"><a href="#cb4-49" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-50"><a href="#cb4-50" aria-hidden="true" tabindex="-1"></a>p_mango_nc <span class="ot">&lt;-</span> dt <span class="sc">|&gt;</span></span>
<span id="cb4-51"><a href="#cb4-51" aria-hidden="true" tabindex="-1"></a>    <span class="fu">count</span>(mango_change) <span class="sc">|&gt;</span></span>
<span id="cb4-52"><a href="#cb4-52" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="at">p =</span> n <span class="sc">/</span> <span class="fu">sum</span>(n)) <span class="sc">|&gt;</span></span>
<span id="cb4-53"><a href="#cb4-53" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(mango_change <span class="sc">!=</span> <span class="st">"No change"</span>) <span class="sc">|&gt;</span></span>
<span id="cb4-54"><a href="#cb4-54" aria-hidden="true" tabindex="-1"></a>    <span class="fu">pull</span>(p)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="overview" class="level1">
<h1>Overview</h1>
<section id="row" class="level2" height="60%">
<h2 height="60%" class="anchored" data-anchor-id="row">Row</h2>
<section id="figure-1" class="level3">
<h3 class="anchored" data-anchor-id="figure-1">Figure 1</h3>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="dashboards/asm.png" class="img-fluid figure-img"></p>
<figcaption><strong>Roles of ASM in neurological disorders</strong> ASM is linked to major neurological disorders such as major depression, Parkinson’s disease, and Alzheimer’s disease.Medications like desipramine and alpha-mangostin, which inhibit ASM and can cross the BBB, show potential for treating neurological disorders. (Modified from Exp Mol Med (2024);56.)</figcaption>
</figure>
</div>
</section>
<section id="column" class="level3" width="55%">
<h3 width="55%" class="anchored" data-anchor-id="column">Column</h3>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="dashboards/lipid_raft.png" class="img-fluid figure-img"></p>
<figcaption><strong>Research Methods</strong> To explore whether ASM inhibition through siASM, desipramine, and alpha-mangostin affect the protein composition of lipid rafts in the same way, we treated cells with or without these ASM inhibitors and analyzed the results using the LC-MS/MS method. This dashboard presents the findings of our quantitative analysis. (Modified from Biomolecules. 2024 Feb; 14(2): 156.)</figcaption>
</figure>
</div>
</section>
</section>
</section>
<section id="results" class="level1">
<h1>Results</h1>
<section id="row-1" class="level2" height="30%">
<h2 height="30%" class="anchored" data-anchor-id="row-1">Row</h2>
<div class="cell" title="Fig 1. Ontology">
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a>prop <span class="ot">&lt;-</span> </span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a>  dt <span class="sc">|&gt;</span></span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>(prot_function) <span class="sc">|&gt;</span></span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">p =</span> <span class="dv">100</span><span class="sc">*</span> n <span class="sc">/</span> <span class="fu">sum</span>(n))</span>
<span id="cb5-5"><a href="#cb5-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-6"><a href="#cb5-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-7"><a href="#cb5-7" aria-hidden="true" tabindex="-1"></a>prop <span class="sc">|&gt;</span></span>
<span id="cb5-8"><a href="#cb5-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(<span class="at">x =</span> <span class="dv">2</span>, <span class="at">y =</span> p, <span class="at">fill =</span> prot_function )) <span class="sc">+</span></span>
<span id="cb5-9"><a href="#cb5-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_col</span>(<span class="at">color =</span> <span class="st">"white"</span>) <span class="sc">+</span></span>
<span id="cb5-10"><a href="#cb5-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="cn">NULL</span>, <span class="at">y =</span> <span class="cn">NULL</span>) <span class="sc">+</span></span>
<span id="cb5-11"><a href="#cb5-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_text</span>(<span class="fu">aes</span>(<span class="at">label =</span> n),</span>
<span id="cb5-12"><a href="#cb5-12" aria-hidden="true" tabindex="-1"></a>             <span class="at">position =</span> <span class="fu">position_stack</span>(<span class="at">vjust =</span> <span class="fl">0.5</span>)) <span class="sc">+</span></span>
<span id="cb5-13"><a href="#cb5-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">coord_polar</span>(<span class="st">"y"</span>, <span class="at">start=</span><span class="dv">0</span>) <span class="sc">+</span></span>
<span id="cb5-14"><a href="#cb5-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_brewer</span>(<span class="at">palette =</span> <span class="st">"GnBu"</span>) <span class="sc">+</span></span>
<span id="cb5-15"><a href="#cb5-15" aria-hidden="true" tabindex="-1"></a>  <span class="co">#ggtitle("A total of 870 Proteins identified in ASM-sensitive DRM") +</span></span>
<span id="cb5-16"><a href="#cb5-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span></span>
<span id="cb5-17"><a href="#cb5-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlim</span>(<span class="fu">c</span>(<span class="fl">0.2</span>, <span class="fl">2.5</span>)) <span class="sc">+</span></span>
<span id="cb5-18"><a href="#cb5-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb5-19"><a href="#cb5-19" aria-hidden="true" tabindex="-1"></a>        <span class="at">panel.grid =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb5-20"><a href="#cb5-20" aria-hidden="true" tabindex="-1"></a>        <span class="co">#axis.title = element_blank(),</span></span>
<span id="cb5-21"><a href="#cb5-21" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.ticks =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb5-22"><a href="#cb5-22" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.text =</span> <span class="fu">element_blank</span>())<span class="sc">+</span></span>
<span id="cb5-23"><a href="#cb5-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">title =</span> <span class="st">"Protein Function </span><span class="sc">\n</span><span class="st">of 870 identified</span><span class="sc">\n</span><span class="st">DRM proteins:"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="asm1_files/figure-html/unnamed-chunk-5-1.png" class="img-fluid figure-img" width="960"></p>
</figure>
</div>
</div>
</div>
<div class="cell" title="Fig 2. siASM and ASM inhibitors">
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="co"># List of items</span></span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a>x <span class="ot">&lt;-</span> <span class="fu">list</span>(<span class="at">siASM =</span> <span class="fu">paste0</span>(dt<span class="sc">$</span>name,dt<span class="sc">$</span>siRNA_change), </span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a>          <span class="at">Desipramine =</span> <span class="fu">paste0</span>(dt<span class="sc">$</span>name,dt<span class="sc">$</span>desi_change), </span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a>          <span class="at">Mangostin  =</span> <span class="fu">paste0</span>(dt<span class="sc">$</span>name,dt<span class="sc">$</span>mango_change))</span>
<span id="cb6-5"><a href="#cb6-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-6"><a href="#cb6-6" aria-hidden="true" tabindex="-1"></a><span class="co"># Venn diagram with custom colors</span></span>
<span id="cb6-7"><a href="#cb6-7" aria-hidden="true" tabindex="-1"></a><span class="fu">ggVennDiagram</span>(x) <span class="sc">+</span> </span>
<span id="cb6-8"><a href="#cb6-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_gradient</span>(<span class="at">low =</span> <span class="st">"#F4FAFE"</span>, <span class="at">high =</span> <span class="st">"#4981BF"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="asm1_files/figure-html/unnamed-chunk-6-1.png" class="img-fluid figure-img" width="960"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="row-2" class="level2" height="30%">
<h2 height="30%" class="anchored" data-anchor-id="row-2">Row</h2>
<div class="cell" title="Fig 3.  protein changes">
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a>desi <span class="ot">&lt;-</span> dt <span class="sc">|&gt;</span></span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>(desi_change) <span class="sc">|&gt;</span></span>
<span id="cb7-3"><a href="#cb7-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">p =</span> n <span class="sc">/</span> <span class="fu">sum</span>(n))<span class="sc">|&gt;</span></span>
<span id="cb7-4"><a href="#cb7-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Treatment =</span> <span class="st">"Desi"</span>)</span>
<span id="cb7-5"><a href="#cb7-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-6"><a href="#cb7-6" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(desi)[<span class="dv">1</span>] <span class="ot">&lt;-</span> <span class="st">"siRNA_change"</span></span>
<span id="cb7-7"><a href="#cb7-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-8"><a href="#cb7-8" aria-hidden="true" tabindex="-1"></a>mango <span class="ot">&lt;-</span> dt <span class="sc">|&gt;</span></span>
<span id="cb7-9"><a href="#cb7-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>(mango_change) <span class="sc">|&gt;</span></span>
<span id="cb7-10"><a href="#cb7-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">p =</span> n <span class="sc">/</span> <span class="fu">sum</span>(n))<span class="sc">|&gt;</span></span>
<span id="cb7-11"><a href="#cb7-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Treatment =</span> <span class="st">"Mango"</span>)</span>
<span id="cb7-12"><a href="#cb7-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-13"><a href="#cb7-13" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(mango)[<span class="dv">1</span>] <span class="ot">&lt;-</span> <span class="st">"siRNA_change"</span></span>
<span id="cb7-14"><a href="#cb7-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-15"><a href="#cb7-15" aria-hidden="true" tabindex="-1"></a>desi_siRNA <span class="ot">&lt;-</span> <span class="fu">rbind</span>(desi,mango,dt <span class="sc">|&gt;</span></span>
<span id="cb7-16"><a href="#cb7-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>(siRNA_change) <span class="sc">|&gt;</span></span>
<span id="cb7-17"><a href="#cb7-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">p =</span> n <span class="sc">/</span> <span class="fu">sum</span>(n))<span class="sc">|&gt;</span></span>
<span id="cb7-18"><a href="#cb7-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Treatment =</span> <span class="st">"siASM"</span>))  </span>
<span id="cb7-19"><a href="#cb7-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-20"><a href="#cb7-20" aria-hidden="true" tabindex="-1"></a>desi_siRNA <span class="sc">|&gt;</span></span>
<span id="cb7-21"><a href="#cb7-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(<span class="at">x =</span> siRNA_change, <span class="at">y =</span> n, <span class="at">group =</span> Treatment, <span class="at">color =</span> Treatment )) <span class="sc">+</span></span>
<span id="cb7-22"><a href="#cb7-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-23"><a href="#cb7-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="at">linewidth =</span> <span class="dv">1</span>) <span class="sc">+</span></span>
<span id="cb7-24"><a href="#cb7-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">size =</span> <span class="dv">2</span>, <span class="at">color =</span> <span class="st">"#0e263560"</span>) <span class="sc">+</span></span>
<span id="cb7-25"><a href="#cb7-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">size =</span> <span class="dv">4</span>, <span class="at">color =</span> <span class="st">"#0e263560"</span>, <span class="at">shape =</span> <span class="st">"circle open"</span>) <span class="sc">+</span></span>
<span id="cb7-26"><a href="#cb7-26" aria-hidden="true" tabindex="-1"></a>    <span class="fu">labs</span>(<span class="at">x =</span> <span class="cn">NULL</span>) <span class="sc">+</span></span>
<span id="cb7-27"><a href="#cb7-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(</span>
<span id="cb7-28"><a href="#cb7-28" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Count"</span>,</span>
<span id="cb7-29"><a href="#cb7-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">sec.axis =</span> <span class="fu">sec_axis</span>(<span class="sc">~</span> . <span class="sc">/</span> <span class="fu">nrow</span>(dt), <span class="at">name =</span> <span class="st">"Proportion"</span>, <span class="at">labels =</span> <span class="fu">label_percent</span>())</span>
<span id="cb7-30"><a href="#cb7-30" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call(C_stringMetric, as.graphicsAnnot(x$label)): font family
not found in Windows font database
Warning in grid.Call(C_stringMetric, as.graphicsAnnot(x$label)): font family
not found in Windows font database</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : font
family not found in Windows font database
Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : font
family not found in Windows font database</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database
Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database
Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database
Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database
Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database
Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database
Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database</code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="asm1_files/figure-html/unnamed-chunk-7-1.png" class="img-fluid figure-img" width="960"></p>
</figure>
</div>
</div>
</div>
<section id="column-1" class="level3" width="40%">
<h3 width="40%" class="anchored" data-anchor-id="column-1">Column</h3>
</section>
<section id="column-2" class="level3 tabset">
<h3 class="tabset anchored" data-anchor-id="column-2">Column</h3>
<div class="cell" title="Fig 4a. siASM">
<div class="sourceCode cell-code" id="cb11"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a>  dt <span class="sc">|&gt;</span></span>
<span id="cb11-2"><a href="#cb11-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>(siRNA_change, prot_function)<span class="sc">|&gt;</span></span>
<span id="cb11-3"><a href="#cb11-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(<span class="at">x =</span> n, <span class="at">y =</span> <span class="fu">fct_rev</span>(prot_function), <span class="at">fill =</span> siRNA_change)) <span class="sc">+</span></span>
<span id="cb11-4"><a href="#cb11-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_col</span>(<span class="at">position =</span> <span class="st">"fill"</span>, <span class="at">color =</span> <span class="st">"white"</span>) <span class="sc">+</span></span>
<span id="cb11-5"><a href="#cb11-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(</span>
<span id="cb11-6"><a href="#cb11-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">values =</span> <span class="fu">c</span>( <span class="st">"pink"</span>,<span class="st">"steelblue"</span>,<span class="st">"#ae8b2d"</span>),</span>
<span id="cb11-7"><a href="#cb11-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">guide =</span> <span class="fu">guide_legend</span>(<span class="at">reverse =</span> <span class="cn">TRUE</span>) </span>
<span id="cb11-8"><a href="#cb11-8" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb11-9"><a href="#cb11-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_continuous</span>(<span class="at">labels =</span> <span class="fu">label_percent</span>()) <span class="sc">+</span></span>
<span id="cb11-10"><a href="#cb11-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">y =</span> <span class="cn">NULL</span>, <span class="at">x =</span> <span class="cn">NULL</span>, <span class="at">fill =</span> <span class="st">"siASM"</span>)<span class="sc">+</span></span>
<span id="cb11-11"><a href="#cb11-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">title =</span> <span class="st">"siASM</span><span class="sc">\n</span><span class="st">sensitive</span><span class="sc">\n</span><span class="st">proteins"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : font
family not found in Windows font database
Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : font
family not found in Windows font database</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database
Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database
Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database</code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="asm1_files/figure-html/unnamed-chunk-8-1.png" class="img-fluid figure-img" width="960"></p>
</figure>
</div>
</div>
</div>
<div class="cell" title="Fig 4b.Desipramine">
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a>  dt <span class="sc">|&gt;</span></span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>(desi_change, prot_function)<span class="sc">|&gt;</span></span>
<span id="cb14-3"><a href="#cb14-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(<span class="at">x =</span> n, <span class="at">y =</span> <span class="fu">fct_rev</span>(prot_function), <span class="at">fill =</span> desi_change)) <span class="sc">+</span></span>
<span id="cb14-4"><a href="#cb14-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_col</span>(<span class="at">position =</span> <span class="st">"fill"</span>, <span class="at">color =</span> <span class="st">"white"</span>) <span class="sc">+</span></span>
<span id="cb14-5"><a href="#cb14-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(</span>
<span id="cb14-6"><a href="#cb14-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">values =</span> <span class="fu">c</span>( <span class="st">"pink"</span>,<span class="st">"steelblue"</span>,<span class="st">"#ae8b2d"</span>),</span>
<span id="cb14-7"><a href="#cb14-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">guide =</span> <span class="fu">guide_legend</span>(<span class="at">reverse =</span> <span class="cn">TRUE</span>)</span>
<span id="cb14-8"><a href="#cb14-8" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb14-9"><a href="#cb14-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_continuous</span>(<span class="at">labels =</span> <span class="fu">label_percent</span>()) <span class="sc">+</span></span>
<span id="cb14-10"><a href="#cb14-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">y =</span> <span class="cn">NULL</span>, <span class="at">x =</span> <span class="cn">NULL</span>, <span class="at">fill =</span> <span class="st">"desi"</span>)<span class="sc">+</span></span>
<span id="cb14-11"><a href="#cb14-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">title =</span> <span class="st">"Desipramine</span><span class="sc">\n</span><span class="st">sensitive</span><span class="sc">\n</span><span class="st">proteins"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : font
family not found in Windows font database
Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : font
family not found in Windows font database</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database
Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database
Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database</code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="asm1_files/figure-html/unnamed-chunk-9-1.png" class="img-fluid figure-img" width="960"></p>
</figure>
</div>
</div>
</div>
<div class="cell" title="Fig 4c.Mangostin">
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a>  dt <span class="sc">|&gt;</span></span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>(mango_change, prot_function)<span class="sc">|&gt;</span></span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(<span class="at">x =</span> n, <span class="at">y =</span> <span class="fu">fct_rev</span>(prot_function), <span class="at">fill =</span> mango_change)) <span class="sc">+</span></span>
<span id="cb17-4"><a href="#cb17-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_col</span>(<span class="at">position =</span> <span class="st">"fill"</span>, <span class="at">color =</span> <span class="st">"white"</span>) <span class="sc">+</span></span>
<span id="cb17-5"><a href="#cb17-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(</span>
<span id="cb17-6"><a href="#cb17-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">values =</span> <span class="fu">c</span>( <span class="st">"pink"</span>,<span class="st">"steelblue"</span>,<span class="st">"#ae8b2d"</span>),</span>
<span id="cb17-7"><a href="#cb17-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">guide =</span> <span class="fu">guide_legend</span>(<span class="at">reverse =</span> <span class="cn">TRUE</span>) </span>
<span id="cb17-8"><a href="#cb17-8" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb17-9"><a href="#cb17-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_continuous</span>(<span class="at">labels =</span> <span class="fu">label_percent</span>()) <span class="sc">+</span></span>
<span id="cb17-10"><a href="#cb17-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">y =</span> <span class="cn">NULL</span>, <span class="at">x =</span> <span class="cn">NULL</span>, <span class="at">fill =</span> <span class="st">"siASM"</span>)<span class="sc">+</span></span>
<span id="cb17-11"><a href="#cb17-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">title =</span> <span class="st">"Mangostin</span><span class="sc">\n</span><span class="st">sensitive</span><span class="sc">\n</span><span class="st">proteins"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : font
family not found in Windows font database
Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : font
family not found in Windows font database</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database
Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database
Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database</code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="asm1_files/figure-html/unnamed-chunk-10-1.png" class="img-fluid figure-img" width="960"></p>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="row-3" class="level2" height="40%">
<h2 height="40%" class="anchored" data-anchor-id="row-3">Row</h2>
<div class="cell" title="Fig 5a.Vesicle related proteins">
<div class="sourceCode cell-code" id="cb20"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a>dt_sub <span class="ot">&lt;-</span> dt <span class="sc">|&gt;</span> <span class="co">#esicle-a</span></span>
<span id="cb20-2"><a href="#cb20-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="fu">str_detect</span>(prot_func, <span class="st">"esicle-"</span>)) <span class="sc">|&gt;</span></span>
<span id="cb20-3"><a href="#cb20-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(change))</span>
<span id="cb20-4"><a href="#cb20-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-5"><a href="#cb20-5" aria-hidden="true" tabindex="-1"></a>dt_sub <span class="ot">&lt;-</span> <span class="fu">rbind</span>(dt_sub[<span class="dv">1</span><span class="sc">:</span><span class="dv">2</span>,],dt_sub[(<span class="fu">nrow</span>(dt_sub)<span class="sc">-</span><span class="dv">1</span>)<span class="sc">:</span>(<span class="fu">nrow</span>(dt_sub)),]) <span class="sc">|&gt;</span> </span>
<span id="cb20-6"><a href="#cb20-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">name =</span> <span class="fu">str_replace</span>(name,<span class="st">"Vesicle-trafficking protein"</span>,<span class="st">""</span>))<span class="sc">|&gt;</span></span>
<span id="cb20-7"><a href="#cb20-7" aria-hidden="true" tabindex="-1"></a><span class="fu">mutate</span>(<span class="at">name =</span> <span class="fu">str_replace</span>(name,<span class="st">"Isoform 1 of "</span>,<span class="st">""</span>))<span class="sc">|&gt;</span></span>
<span id="cb20-8"><a href="#cb20-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">name =</span> <span class="fu">str_replace</span>(name,<span class="st">"Vesicle-associated membrane protein "</span>, <span class="st">""</span>))</span>
<span id="cb20-9"><a href="#cb20-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-10"><a href="#cb20-10" aria-hidden="true" tabindex="-1"></a>dt_sub[ , <span class="dv">1</span><span class="sc">:</span><span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">str_split_fixed</span>(dt_sub<span class="sc">$</span>name, <span class="st">","</span>, <span class="dv">2</span>)</span>
<span id="cb20-11"><a href="#cb20-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-12"><a href="#cb20-12" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(dt_sub)[<span class="dv">1</span><span class="sc">:</span><span class="dv">5</span>] <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"name"</span>,<span class="st">'x'</span>, <span class="st">"siRNA"</span>,<span class="st">"Desipramine"</span>,<span class="st">"Mangosteen"</span>)</span>
<span id="cb20-13"><a href="#cb20-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-14"><a href="#cb20-14" aria-hidden="true" tabindex="-1"></a>dt_sub <span class="ot">&lt;-</span> <span class="fu">melt</span>(<span class="fu">data.frame</span>(dt_sub[,<span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">3</span><span class="sc">:</span><span class="dv">5</span>)]),<span class="at">id.vars=</span><span class="st">"name"</span>, <span class="at">measured.vars=</span><span class="fu">c</span>(<span class="st">"change"</span>, <span class="st">"change_desi"</span>,<span class="st">"change_mango"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: The melt generic in data.table has been passed a data.frame and will
attempt to redirect to the relevant reshape2 method; please note that reshape2
is superseded and is no longer actively developed, and this redirection is now
deprecated. To continue using melt methods from reshape2 while both libraries
are attached, e.g. melt.list, you can prepend the namespace, i.e.
reshape2::melt(data.frame(dt_sub[, c(1, 3:5)])). In the next version, this
warning will become an error.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb22"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a>dt_sub <span class="sc">|&gt;</span></span>
<span id="cb22-2"><a href="#cb22-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">value =</span> <span class="fu">ifelse</span>(value <span class="sc">&gt;=</span><span class="dv">1</span> , <span class="sc">-</span><span class="dv">1</span><span class="sc">*</span>value, <span class="dv">1</span><span class="sc">/</span>value))<span class="sc">|&gt;</span></span>
<span id="cb22-3"><a href="#cb22-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(value))<span class="sc">|&gt;</span></span>
<span id="cb22-4"><a href="#cb22-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(<span class="at">fill=</span> variable, <span class="at">y=</span>value, <span class="at">x=</span>name)) <span class="sc">+</span> </span>
<span id="cb22-5"><a href="#cb22-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_bar</span>(<span class="at">position=</span><span class="st">'dodge'</span>, <span class="at">stat=</span><span class="st">'identity'</span>)<span class="sc">+</span> </span>
<span id="cb22-6"><a href="#cb22-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">coord_flip</span>() <span class="sc">+</span></span>
<span id="cb22-7"><a href="#cb22-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">labs</span>(<span class="at">y =</span> <span class="cn">NULL</span>, <span class="at">x =</span> <span class="cn">NULL</span>)<span class="sc">+</span></span>
<span id="cb22-8"><a href="#cb22-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_fill_manual</span>(<span class="st">''</span>, <span class="at">values=</span><span class="fu">c</span>(<span class="st">'coral2'</span>, <span class="st">'steelblue'</span>, <span class="st">'#ae8b2d'</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : font
family not found in Windows font database</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : font
family not found in Windows font database</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database</code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="asm1_files/figure-html/unnamed-chunk-11-1.png" class="img-fluid figure-img" width="960"></p>
</figure>
</div>
</div>
</div>
<div class="cell" title="Fig 5b. Apoptosis">
<div class="sourceCode cell-code" id="cb26"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a>dt_sub <span class="ot">&lt;-</span> dt <span class="sc">|&gt;</span> </span>
<span id="cb26-2"><a href="#cb26-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="fu">str_detect</span>(prot_func, <span class="st">"poptosis"</span>)) <span class="sc">|&gt;</span></span>
<span id="cb26-3"><a href="#cb26-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(change))</span>
<span id="cb26-4"><a href="#cb26-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-5"><a href="#cb26-5" aria-hidden="true" tabindex="-1"></a>dt_sub <span class="ot">&lt;-</span> <span class="fu">rbind</span>(dt_sub[<span class="dv">1</span><span class="sc">:</span><span class="dv">2</span>,],dt_sub[(<span class="fu">nrow</span>(dt_sub)<span class="sc">-</span><span class="dv">1</span>)<span class="sc">:</span>(<span class="fu">nrow</span>(dt_sub)),]) <span class="sc">|&gt;</span> </span>
<span id="cb26-6"><a href="#cb26-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">name =</span> <span class="fu">str_replace</span>(name,<span class="st">"Isoform 4 of Inhibitor of nuclear factor kappa-B kinase-interacting protein"</span>,<span class="st">"IKBIP"</span>))<span class="sc">|&gt;</span></span>
<span id="cb26-7"><a href="#cb26-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">name =</span> <span class="fu">str_replace</span>(name,<span class="st">"Carboxypeptidase D"</span>, <span class="st">"CPD"</span>)) <span class="sc">|&gt;</span></span>
<span id="cb26-8"><a href="#cb26-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">name =</span> <span class="fu">str_replace</span>(name,<span class="st">"T-complex protein 1 subunit eta"</span>, <span class="st">"CCT7"</span>)) <span class="sc">|&gt;</span></span>
<span id="cb26-9"><a href="#cb26-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">name =</span> <span class="fu">str_replace</span>(name,<span class="st">"Guanine nucleotide-binding protein G"</span>, <span class="st">"GNG12 ,"</span>))</span>
<span id="cb26-10"><a href="#cb26-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-11"><a href="#cb26-11" aria-hidden="true" tabindex="-1"></a>dt_sub[ , <span class="dv">1</span><span class="sc">:</span><span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">str_split_fixed</span>(dt_sub<span class="sc">$</span>name, <span class="st">","</span>, <span class="dv">2</span>)</span>
<span id="cb26-12"><a href="#cb26-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-13"><a href="#cb26-13" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(dt_sub)[<span class="dv">1</span><span class="sc">:</span><span class="dv">5</span>] <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"name"</span>,<span class="st">'x'</span>, <span class="st">"siRNA"</span>,<span class="st">"Desipramine"</span>,<span class="st">"Mangosteen"</span>)</span>
<span id="cb26-14"><a href="#cb26-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-15"><a href="#cb26-15" aria-hidden="true" tabindex="-1"></a>dt_sub <span class="ot">&lt;-</span> <span class="fu">melt</span>(<span class="fu">data.frame</span>(dt_sub[,<span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">3</span><span class="sc">:</span><span class="dv">5</span>)]),<span class="at">id.vars=</span><span class="st">"name"</span>, <span class="at">measured.vars=</span><span class="fu">c</span>(<span class="st">"change"</span>, <span class="st">"change_desi"</span>,<span class="st">"change_mango"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: The melt generic in data.table has been passed a data.frame and will
attempt to redirect to the relevant reshape2 method; please note that reshape2
is superseded and is no longer actively developed, and this redirection is now
deprecated. To continue using melt methods from reshape2 while both libraries
are attached, e.g. melt.list, you can prepend the namespace, i.e.
reshape2::melt(data.frame(dt_sub[, c(1, 3:5)])). In the next version, this
warning will become an error.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb28"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb28-1"><a href="#cb28-1" aria-hidden="true" tabindex="-1"></a>dt_sub <span class="sc">|&gt;</span></span>
<span id="cb28-2"><a href="#cb28-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">value =</span> <span class="fu">ifelse</span>(value <span class="sc">&gt;=</span><span class="dv">1</span> , <span class="sc">-</span><span class="dv">1</span><span class="sc">*</span>value, <span class="dv">1</span><span class="sc">/</span>value))<span class="sc">|&gt;</span></span>
<span id="cb28-3"><a href="#cb28-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(<span class="at">fill=</span> variable, <span class="at">y=</span>value, <span class="at">x=</span>name)) <span class="sc">+</span> </span>
<span id="cb28-4"><a href="#cb28-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_bar</span>(<span class="at">position=</span><span class="st">'dodge'</span>, <span class="at">stat=</span><span class="st">'identity'</span>)<span class="sc">+</span> </span>
<span id="cb28-5"><a href="#cb28-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">coord_flip</span>() <span class="sc">+</span></span>
<span id="cb28-6"><a href="#cb28-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="cn">NULL</span>) <span class="sc">+</span> </span>
<span id="cb28-7"><a href="#cb28-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="cn">NULL</span>)<span class="sc">+</span></span>
<span id="cb28-8"><a href="#cb28-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_fill_manual</span>(<span class="st">''</span>, <span class="at">values=</span><span class="fu">c</span>(<span class="st">'coral2'</span>, <span class="st">'steelblue'</span>, <span class="st">'#ae8b2d'</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : font
family not found in Windows font database</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : font
family not found in Windows font database</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database</code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="asm1_files/figure-html/unnamed-chunk-12-1.png" class="img-fluid figure-img" width="960"></p>
</figure>
</div>
</div>
</div>
<div class="cell" title="Fig 5c.Enzymes">
<div class="sourceCode cell-code" id="cb32"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb32-1"><a href="#cb32-1" aria-hidden="true" tabindex="-1"></a>dt_sub <span class="ot">&lt;-</span> dt <span class="sc">|&gt;</span> <span class="co">#esicle-a</span></span>
<span id="cb32-2"><a href="#cb32-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="fu">str_detect</span>(prot_func, <span class="st">"ase"</span>)) <span class="sc">|&gt;</span></span>
<span id="cb32-3"><a href="#cb32-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(change))</span>
<span id="cb32-4"><a href="#cb32-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb32-5"><a href="#cb32-5" aria-hidden="true" tabindex="-1"></a>dt_sub <span class="ot">&lt;-</span> <span class="fu">rbind</span>(dt_sub[<span class="dv">1</span><span class="sc">:</span><span class="dv">2</span>,],dt_sub[(<span class="fu">nrow</span>(dt_sub)<span class="sc">-</span><span class="dv">1</span>)<span class="sc">:</span>(<span class="fu">nrow</span>(dt_sub)),]) <span class="sc">|&gt;</span> </span>
<span id="cb32-6"><a href="#cb32-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">name =</span> <span class="fu">str_replace</span>(name,<span class="st">"CD73, 5'-nucleotidase"</span>,<span class="st">"5'-nucleotidase"</span>))<span class="sc">|&gt;</span></span>
<span id="cb32-7"><a href="#cb32-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">name =</span> <span class="fu">str_replace</span>(name,<span class="st">"ATP synthase subunit alpha, mitochondrial"</span>, <span class="st">"ATP5F1E"</span>)) <span class="sc">|&gt;</span></span>
<span id="cb32-8"><a href="#cb32-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">name =</span> <span class="fu">str_replace</span>(name,<span class="st">"Isoform alpha-enolase of Alpha-enolase"</span>, <span class="st">"ENO1"</span>))</span>
<span id="cb32-9"><a href="#cb32-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb32-10"><a href="#cb32-10" aria-hidden="true" tabindex="-1"></a>dt_sub[ , <span class="dv">1</span><span class="sc">:</span><span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">str_split_fixed</span>(dt_sub<span class="sc">$</span>name, <span class="st">","</span>, <span class="dv">2</span>)</span>
<span id="cb32-11"><a href="#cb32-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb32-12"><a href="#cb32-12" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(dt_sub)[<span class="dv">1</span><span class="sc">:</span><span class="dv">5</span>] <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"name"</span>,<span class="st">'x'</span>, <span class="st">"siRNA"</span>,<span class="st">"Desipramine"</span>,<span class="st">"Mangosteen"</span>)</span>
<span id="cb32-13"><a href="#cb32-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb32-14"><a href="#cb32-14" aria-hidden="true" tabindex="-1"></a>dt_sub <span class="ot">&lt;-</span> <span class="fu">melt</span>(<span class="fu">data.frame</span>(dt_sub[,<span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">3</span><span class="sc">:</span><span class="dv">5</span>)]),<span class="at">id.vars=</span><span class="st">"name"</span>, <span class="at">measured.vars=</span><span class="fu">c</span>(<span class="st">"change"</span>, <span class="st">"change_desi"</span>,<span class="st">"change_mango"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: The melt generic in data.table has been passed a data.frame and will
attempt to redirect to the relevant reshape2 method; please note that reshape2
is superseded and is no longer actively developed, and this redirection is now
deprecated. To continue using melt methods from reshape2 while both libraries
are attached, e.g. melt.list, you can prepend the namespace, i.e.
reshape2::melt(data.frame(dt_sub[, c(1, 3:5)])). In the next version, this
warning will become an error.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb34"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb34-1"><a href="#cb34-1" aria-hidden="true" tabindex="-1"></a>dt_sub <span class="sc">|&gt;</span></span>
<span id="cb34-2"><a href="#cb34-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">value =</span> <span class="fu">ifelse</span>(value <span class="sc">&gt;=</span><span class="dv">1</span> , <span class="sc">-</span><span class="dv">1</span><span class="sc">*</span>value, <span class="dv">1</span><span class="sc">/</span>value))<span class="sc">|&gt;</span></span>
<span id="cb34-3"><a href="#cb34-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(<span class="at">fill=</span> variable, <span class="at">y=</span>value, <span class="at">x=</span>name)) <span class="sc">+</span> </span>
<span id="cb34-4"><a href="#cb34-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_bar</span>(<span class="at">position=</span><span class="st">'dodge'</span>, <span class="at">stat=</span><span class="st">'identity'</span>)<span class="sc">+</span> </span>
<span id="cb34-5"><a href="#cb34-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">coord_flip</span>() <span class="sc">+</span></span>
<span id="cb34-6"><a href="#cb34-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="cn">NULL</span>) <span class="sc">+</span> </span>
<span id="cb34-7"><a href="#cb34-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="cn">NULL</span>)<span class="sc">+</span></span>
<span id="cb34-8"><a href="#cb34-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(<span class="st">''</span>, <span class="at">values=</span><span class="fu">c</span>(<span class="st">'coral2'</span>, <span class="st">'steelblue'</span>, <span class="st">'#ae8b2d'</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : font
family not found in Windows font database</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : font
family not found in Windows font database</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y, :
font family not found in Windows font database</code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="asm1_files/figure-html/unnamed-chunk-13-1.png" class="img-fluid figure-img" width="960"></p>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="data" class="level1">
<h1>Data</h1>
<div class="cell">
<div class="sourceCode cell-code" id="cb38"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb38-1"><a href="#cb38-1" aria-hidden="true" tabindex="-1"></a>dt[,<span class="fu">c</span>(<span class="dv">2</span><span class="sc">:</span><span class="dv">5</span>)] <span class="sc">|&gt;</span></span>
<span id="cb38-2"><a href="#cb38-2" aria-hidden="true" tabindex="-1"></a><span class="co">#  arrange(id) |&gt;</span></span>
<span id="cb38-3"><a href="#cb38-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">datatable</span>(</span>
<span id="cb38-4"><a href="#cb38-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Protein Name"</span>,<span class="st">"Change in siASM"</span>, <span class="st">"Change in Desi"</span>, <span class="st">"Change in Mangosteen"</span>),</span>
<span id="cb38-5"><a href="#cb38-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">options =</span> <span class="fu">list</span>(<span class="at">dom =</span> <span class="st">'ftp'</span>, <span class="at">paging =</span> <span class="cn">TRUE</span>)</span>
<span id="cb38-6"><a href="#cb38-6" aria-hidden="true" tabindex="-1"></a>    )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-71ff4803cd16abd171b3" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-71ff4803cd16abd171b3">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432","433","434","435","436","437","438","439","440","441","442","443","444","445","446","447","448","449","450","451","452","453","454","455","456","457","458","459","460","461","462","463","464","465","466","467","468","469","470","471","472","473","474","475","476","477","478","479","480","481","482","483","484","485","486","487","488","489","490","491","492","493","494","495","496","497","498","499","500","501","502","503","504","505","506","507","508","509","510","511","512","513","514","515","516","517","518","519","520","521","522","523","524","525","526","527","528","529","530","531","532","533","534","535","536","537","538","539","540","541","542","543","544","545","546","547","548","549","550","551","552","553","554","555","556","557","558","559","560","561","562","563","564","565","566","567","568","569","570","571","572","573","574","575","576","577","578","579","580","581","582","583","584","585","586","587","588","589","590","591","592","593","594","595","596","597","598","599","600","601","602","603","604","605","606","607","608","609","610","611","612","613","614","615","616","617","618","619","620","621","622","623","624","625","626","627","628","629","630","631","632","633","634","635","636","637","638","639","640","641","642","643","644","645","646","647","648","649","650","651","652","653","654","655","656","657","658","659","660","661","662","663","664","665","666","667","668","669","670","671","672","673","674","675","676","677","678","679","680","681","682","683","684","685","686","687","688","689","690","691","692","693","694","695","696","697","698","699","700","701","702","703","704","705","706","707","708","709","710","711","712","713","714","715","716","717","718","719","720","721","722","723","724","725","726","727","728","729","730","731","732","733","734","735","736","737","738","739","740","741","742","743","744","745","746","747","748","749","750","751","752","753","754","755","756","757","758","759","760","761","762","763","764","765","766","767","768","769","770","771","772","773","774","775","776","777","778","779","780","781","782","783","784","785","786","787","788","789","790","791","792","793","794","795","796","797","798","799","800","801","802","803","804","805","806","807","808","809","810","811","812","813","814","815","816","817","818","819","820","821","822","823","824","825","826","827","828","829","830","831","832","833","834","835"],["CD73, 5'-nucleotidase","ATP synthase subunit alpha, mitochondrial","Beta-2-microglobulin","CD59 glycoprotein","Complement component 1 Q subcomponent-binding protein, mitochondrial","Isoform Alpha of Caveolin-1","Putative uncharacterized protein ZNF326","Thy-1 membrane glycoprotein","VAMP3, Vesicle-associated membrane protein 3","Vesicle-trafficking protein SEC22b","104 kDa protein","13 kDa protein","13kDa differentiation-associated protein variant (Fragment)","14-3-3 protein epsilon","14-3-3 protein gamma","14-3-3 protein theta","14-3-3 protein zeta/delta","165 kDa protein","20 kDa protein","28S ribosomal protein S31, mitochondrial","29 kDa protein","30 kDa protein","33 kDa protein","39S ribosomal protein L1, mitochondrial","39S ribosomal protein L48, mitochondrial","3-hydroxyacyl-CoA dehydratase 3","40S ribosomal protein S18","40S ribosomal protein S3","40S ribosomal protein S4, X isoform","40S ribosomal protein S8","44 kDa protein","475 kDa protein","51 kDa protein","51 kDa protein","59 kDa protein","60 kDa heat shock protein, mitochondrial","60S acidic ribosomal protein P0","60S acidic ribosomal protein P2","60S ribosomal protein L13","60S ribosomal protein L18","60S ribosomal protein L23","60S ribosomal protein L4","60S ribosomal protein L6","7-dehydrocholesterol reductase","91 kDa protein","Actin, cytoplasmic 1","Acyl carrier protein, mitochondrial","Acyl-CoA dehydrogenase family member 9, mitochondrial","ADP/ATP translocase 1","ADP/ATP translocase 2","ADP/ATP translocase 3","AFG3-like protein 2","A-kinase anchor protein 6","Alpha-actinin-2","Alpha-actinin-4","Alpha-mannosidase 2","Alpha-soluble NSF attachment protein","Ankyrin repeat and SOCS box protein 15","Annexin A1","annexin A4","Annexin A5","Annexin A6","Antigen peptide transporter 1","antigen peptide transporter 2 isoform 1","AP-1 complex subunit gamma-1 isoform a","Apolipoprotein B-100","Aspartate aminotransferase, mitochondrial","ATP synthase subunit b, mitochondrial","ATP synthase subunit beta, mitochondrial","ATP synthase subunit delta, mitochondrial","ATP synthase subunit g, mitochondrial","ATP synthase subunit O, mitochondrial","ATP synthase, H+ transporting, mitochondrial F0 complex, subunit E","ATP synthase-coupling factor 6, mitochondrial","Baculoviral IAP repeat-containing protein 6","Band 4.1-like protein 2","Basal cell adhesion molecule","B-cell receptor-associated protein 31","beta-adducin isoform c","Bile salt export pump","Brain-specific angiogenesis inhibitor 3","Breast cancer type 2 susceptibility protein","CAAX prenyl protease 1 homolog","Cadherin-2","Calcium signal-modulating cyclophilin ligand","Calcium-binding mitochondrial carrier protein Aralar1","Calcium-binding mitochondrial carrier protein Aralar2","Calcyphosin","Calmodulin","Calmodulin-like protein 5","Calpain-1 catalytic subunit","Carbonic anhydrase 9","Carboxypeptidase D","Carnitine O-palmitoyltransferase 2, mitochondrial","Cathepsin D","Cation-dependent mannose-6-phosphate receptor","Cation-independent mannose-6-phosphate receptor","Cd002 protein","CD151 antigen","CD70 antigen","CD81 antigen","CD82 antigen","CD9 antigen","cDNA FLJ14473 fis, clone MAMMA1001080, highly similar to Homo sapiens SNC73 protein (SNC73) mRNA","cDNA FLJ45706 fis, clone FEBRA2028457, highly similar to Nucleolin","cDNA FLJ51518, highly similar to Annexin A11","cDNA FLJ51775, highly similar to 14-3-3 protein zeta/delta","cDNA FLJ52398, highly similar to Cadherin-13","cDNA FLJ54408, highly similar to Heat shock 70 kDa protein 1","cDNA FLJ54957, highly similar to Transketolase","cDNA FLJ55177, highly similar to Ras-related protein Ral-B","cDNA FLJ55574, highly similar to Calnexin","cDNA FLJ56074, highly similar to 150 kDa oxygen-regulated protein","cDNA FLJ56157, highly similar to Glucosylceramidase","cDNA FLJ56280, highly similar to Endoplasmic reticulum-Golgi intermediate compartment protein 1","cDNA FLJ56370, highly similar to Homo sapiens FK506 binding protein 8, 38kDa (FKBP8), mRNA","cDNA FLJ56389, highly similar to Elongation factor 1-gamma","cDNA FLJ56403, highly similar to CDK5 regulatory subunit-associated protein 3","cDNA FLJ56425, highly similar to Very-long-chain specific acyl-CoAdehydrogenase, mitochondrial","cDNA FLJ56478, highly similar to Homo sapiens COBL-like 1 (COBLL1), mRNA","cDNA FLJ56903, highly similar to Tubulin beta-7 chain","cDNA FLJ59191, highly similar to NADH dehydrogenase (ubiquinone) 1 alpha subcomplex subunit 13","cDNA FLJ59612, highly similar to Lactadherin","cDNA FLJ60097, highly similar to Tubulin alpha-ubiquitous chain","cDNA FLJ60124, highly similar to Mitochondrial dicarboxylate carrier","cDNA FLJ60299, highly similar to Rab GDP dissociation inhibitor beta","cDNA FLJ60424, highly similar to Junction plakoglobin","Centromere protein B, DNA-binding region sub-type domain containing protein","Ceruloplasmin","Choline-phosphate cytidylyltransferase A","Chondroitin sulfate proteoglycan 4","Citrate synthase, mitochondrial","Cofilin-1","Coiled-coil-helix-coiled-coil-helix domain-containing protein 3, mitochondrial","Collagen alpha-1(I) chain","Collagen alpha-3(V) chain","Collapsin response mediator protein 4 long variant","Conserved hypothetical protein","Contactin-associated protein 1","COP9 signalosome complex subunit 3","Copine-3","CTTNBP2 N-terminal-like protein","C-type mannose receptor 2","Cytochrome b-c1 complex subunit 1, mitochondrial","Cytochrome b-c1 complex subunit 2, mitochondrial","Cytochrome b-c1 complex subunit 6, mitochondrial","Cytochrome b-c1 complex subunit 7","Cytochrome b-c1 complex subunit Rieske, mitochondrial","Cytochrome c oxidase subunit 2","Cytochrome c oxidase subunit 4 isoform 1, mitochondrial","Cytochrome c oxidase subunit 5A, mitochondrial","Cytochrome c oxidase subunit 5B, mitochondrial","Cytochrome c oxidase subunit 6B1","Cytochrome c oxidase subunit 6C","cytochrome c oxidase subunit VIIa polypeptide 2 (liver) precursor","Cytochrome c1, heme protein, mitochondrial","Cytochrome P450 20A1","Cytochrome P450 monooxygenase","D-3-phosphoglycerate dehydrogenase","Decay-accelerating factor splicing variant 1","Deoxyribonuclease-1-like 1","Derlin-1","Dermcidin","Desmoglein-1","Desmoglein-2","Diacylglycerol O-acyltransferase 1","Dihydrolipoyl dehydrogenase, mitochondrial","Dihydrolipoyllysine-residue acetyltransferase component of pyruvate dehydrogenase complex, mitochondrial","Disintegrin and metalloproteinase domain-containing protein 10","DNA repair protein","DnaJ homolog subfamily C member 13","Dolichol-phosphate mannosyltransferase","Dolichyl-diphosphooligosaccharide--protein glycosyltransferase 48 kDa subunit","Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit 1 precursor","Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit 2","Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit DAD1","Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit STT3A","Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit STT3B","dynein heavy chain 17, axonemal","Dynein heavy chain domain-containing protein 1","Dystroglycan","EH domain-containing protein 4","Elongation factor 1-alpha 1","Elongation factor 2","Emerin","Endoplasmic reticulum mannosyl-oligosaccharide 1,2-alpha-mannosidase","Endoplasmic reticulum metallopeptidase 1","Endoplasmic reticulum resident protein 44","Endoplasmin","Endothelial protein C receptor precursor","Ephrin type-A receptor 2","Epiplakin","Epoxide hydrolase 1","Erythrocyte band 7 integral membrane protein","Eukaryotic initiation factor 4A-I","Ezrin","FAM73A protein","FAS-associated factor 2","Fatty acid-binding protein, epidermal","Filaggrin-2","Flotillin-1","Flotillin-2","Galectin-1","Galectin-3-binding protein","Glutamate receptor 4","Glyceraldehyde-3-phosphate dehydrogenase","Glycogen phosphorylase, brain form","Glypican-1","Glypican-4","Glypican-6","Golgi integral membrane protein 4","Golgi membrane protein 1","Golgi SNAP receptor complex member 1","Golgin subfamily A member 7","Golgin subfamily B member 1","GPI-anchor transamidase","GTPase HRas","GTPase IMAP family member 8","Guanine nucleotide-binding protein G(I)/G(S)/G(O) subunit gamma-12","Guanine nucleotide-binding protein G(I)/G(S)/G(T) subunit beta-1","Guanine nucleotide-binding protein G(I)/G(S)/G(T) subunit beta-2","Guanine nucleotide-binding protein G(k) subunit alpha","Guanine nucleotide-binding protein subunit alpha-11","Guanine nucleotide-binding protein subunit alpha-13","Guanylate cyclase soluble subunit alpha-3","HCG1990625, isoform CRA_a","Heat shock protein beta-1","Heat shock protein HSP 90-beta","Heme oxygenase 2","hepatitis B virus X-interacting protein","Heterogeneous nuclear ribonucleoprotein L","High affinity cationic amino acid transporter 1","Hippocalcin-like protein 1","Histone H1.3","Histone H2A type 1-B/E","Histone H2B type 2-E","Histone H3.2","Histone H4","HLA class I histocompatibility antigen, A-2 alpha chain","HLA class I histocompatibility antigen, B-18 alpha chain","HLA class II histocompatibility antigen, DP alpha 1 chain","HLA class II histocompatibility antigen, DR alpha chain","HLA class II histocompatibility antigen, DRB1-1 beta chain","HNRPU protein","Hornerin","Hsp90 co-chaperone Cdc37","HSPA5 protein","hypothetical protein LOC643677","Importin subunit beta-1","Importin-9","Inositol 1,4,5-trisphosphate receptor type 3","Inositol monophosphatase 3","Integral membrane protein 2B","Integrin alpha-1","Integrin alpha-2","Integrin alpha-5","Integrin alpha-9","integrin alpha-V isoform 2","Integrin beta-8","Interferon-induced transmembrane protein 3","Involucrin","Isoform 1 of Acylglycerol kinase, mitochondrial","Isoform 1 of Adipocyte plasma membrane-associated protein","Isoform 1 of ADP-ribosylation factor GTPase-activating protein 1","Isoform 1 of Amyloid-like protein 2","Isoform 1 of Amyotrophic lateral sclerosis 2 chromosomal region candidate gene 8 protein","Isoform 1 of Annexin A7","Isoform 1 of AP-2 complex subunit beta","Isoform 1 of Apolipoprotein O","Isoform 1 of Apoptosis-inducing factor 1, mitochondrial","Isoform 1 of ATP synthase subunit d, mitochondrial","Isoform 1 of ATPase family AAA domain-containing protein 1","Isoform 1 of ATP-binding cassette sub-family A member 13","Isoform 1 of ATP-binding cassette sub-family B member 7, mitochondrial","Isoform 1 of ATP-binding cassette sub-family D member 3","Isoform 1 of ATP-dependent metalloprotease YME1L1","Isoform 1 of Autophagy-related protein 9A","Isoform 1 of Calcium-binding mitochondrial carrier protein SCaMC-1","Isoform 1 of Canalicular multispecific organic anion transporter 2","Isoform 1 of Catenin alpha-1","Isoform 1 of Catenin beta-1","Isoform 1 of CD109 antigen","Isoform 1 of CD166 antigen","Isoform 1 of CD276 antigen","Isoform 1 of Cell adhesion molecule 1","Isoform 1 of Cell cycle control protein 50A","Isoform 1 of Cell surface glycoprotein MUC18","Isoform 1 of Centrosomal protein of 164 kDa","Isoform 1 of Choline transporter-like protein 2","Isoform 1 of Citron Rho-interacting kinase","Isoform 1 of Clathrin heavy chain 1","Isoform 1 of CLIP-associating protein 1","Isoform 1 of CMP-N-acetylneuraminate-beta-galactosamide-alpha-2,3-sialyltransferase 4","Isoform 1 of Contactin-1","Isoform 1 of Coxsackievirus and adenovirus receptor","Isoform 1 of Cullin-associated NEDD8-dissociated protein 1","Isoform 1 of Cytochrome c oxidase assembly protein COX15 homolog","Isoform 1 of Cytoskeleton-associated protein 4","Isoform 1 of DBH-like monooxygenase protein 1","Isoform 1 of DDRGK domain-containing protein 1","Isoform 1 of DNA repair protein REV1","Isoform 1 of DNA-dependent protein kinase catalytic subunit","Isoform 1 of Dynein heavy chain 7, axonemal","Isoform 1 of E3 ubiquitin-protein ligase CHIP","Isoform 1 of E3 ubiquitin-protein ligase synoviolin","Isoform 1 of E3 UFM1-protein ligase 1","Isoform 1 of EGF-like repeat and discoidin I-like domain-containing protein 3","Isoform 1 of Electron transfer flavoprotein subunit beta","Isoform 1 of Elongation factor 1-delta","Isoform 1 of Endoplasmic reticulum-Golgi intermediate compartment protein 3","Isoform 1 of Ephrin type-A receptor 3","Isoform 1 of Epidermal growth factor receptor","Isoform 1 of Epsin-1","Isoform 1 of Erlin-2","Isoform 1 of Evolutionarily conserved signaling intermediate in Toll pathway, mitochondrial","Isoform 1 of Exportin-2","Isoform 1 of Extended synaptotagmin-1","Isoform 1 of Fanconi anemia group J protein","Isoform 1 of Fibrous sheath-interacting protein 2","Isoform 1 of Filamin-B","Isoform 1 of Fragile X mental retardation 1 protein","Isoform 1 of GDNF family receptor alpha-1","Isoform 1 of Gelsolin","Isoform 1 of Glial fibrillary acidic protein","Isoform 1 of Glycerol-3-phosphate dehydrogenase, mitochondrial","Isoform 1 of Golgi pH regulator A","Isoform 1 of GPI ethanolamine phosphate transferase 3","Isoform 1 of GPI transamidase component PIG-S","Isoform 1 of Guanine nucleotide-binding protein G(i) subunit alpha-2","Isoform 1 of HEAT repeat-containing protein 7B2","Isoform 1 of Heat shock cognate 71 kDa protein","Isoform 1 of Hemicentin-1","Isoform 1 of Heparan sulfate 2-O-sulfotransferase 1","Isoform 1 of Hepatocyte growth factor receptor","Isoform 1 of Heterogeneous nuclear ribonucleoprotein D0","Isoform 1 of Heterogeneous nuclear ribonucleoprotein K","Isoform 1 of Heterogeneous nuclear ribonucleoprotein M","Isoform 1 of Hyaluronan mediated motility receptor","Isoform 1 of Immunoglobulin superfamily member 8","Isoform 1 of Importin-5","Isoform 1 of Keratin, type I cuticular Ha6","Isoform 1 of Keratin, type I cytoskeletal 13","Isoform 1 of Keratin, type II cytoskeletal 78","Isoform 1 of Kinectin","Isoform 1 of Layilin","Isoform 1 of LETM1 and EF-hand domain-containing protein 1, mitochondrial","Isoform 1 of Leucine-rich repeat and IQ domain-containing protein 3","Isoform 1 of Leucyl-cystinyl aminopeptidase","Isoform 1 of Lipid phosphate phosphohydrolase 1","Isoform 1 of Lymphocyte function-associated antigen 3","Isoform 1 of Lysine-specific demethylase 4B","Isoform 1 of Lysocardiolipin acyltransferase 1","Isoform 1 of Lysophospholipid acyltransferase 7","Isoform 1 of Major prion protein","Isoform 1 of Melanotransferrin","Isoform 1 of Metalloreductase STEAP3","Isoform 1 of Minor histocompatibility antigen H13","Isoform 1 of Mitochondrial antiviral-signaling protein","Isoform 1 of Mitochondrial import receptor subunit TOM40 homolog","Isoform 1 of Mitochondrial inner membrane protein","Isoform 1 of Mitochondrial Rho GTPase 2","Isoform 1 of Mitogen-activated protein-binding protein-interacting protein","Isoform 1 of Myoferlin","Isoform 1 of Myosin-9","Isoform 1 of Myosin-Va","Isoform 1 of Na(+)/H(+) exchange regulatory cofactor NHE-RF2","Isoform 1 of NADH dehydrogenase [ubiquinone] flavoprotein 1, mitochondrial","Isoform 1 of NADH-cytochrome b5 reductase 3","Isoform 1 of Natural resistance-associated macrophage protein 2","Isoform 1 of Neuronal cell adhesion molecule","Isoform 1 of Neuroplastin","Isoform 1 of Nicastrin","Isoform 1 of OCIA domain-containing protein 2","Isoform 1 of Oxysterol-binding protein 1","Isoform 1 of Phosphatidate cytidylyltransferase 2","Isoform 1 of Phosphatidylinositol 4-kinase alpha","Isoform 1 of Plexin domain-containing protein 2","Isoform 1 of Plexin-D1","Isoform 1 of Polyadenylate-binding protein 1","Isoform 1 of Polymerase I and transcript release factor","Isoform 1 of Polypeptide N-acetylgalactosaminyltransferase 1","Isoform 1 of POTE ankyrin domain family member E","Isoform 1 of Pre-B-cell leukemia transcription factor-interacting protein 1","Isoform 1 of Protein dpy-19 homolog 1","Isoform 1 of Protein EFR3 homolog A","Isoform 1 of Protein piccolo","Isoform 1 of Protein SET","Isoform 1 of Protein sidekick-2","Isoform 1 of Protocadherin gamma-C3","Isoform 1 of Putative adenosylhomocysteinase 2","Isoform 1 of Putative Polycomb group protein ASXL1","Isoform 1 of Putative SMEK homolog 3","Isoform 1 of Pyruvate dehydrogenase E1 component subunit beta, mitochondrial","Isoform 1 of Ras-related protein Rab-1A","Isoform 1 of Ras-related protein Rab-6A","Isoform 1 of Roundabout homolog 2","Isoform 1 of RUN and FYVE domain-containing protein 4","Isoform 1 of Sacsin","Isoform 1 of Sarcoplasmic/endoplasmic reticulum calcium ATPase 2","Isoform 1 of Secretory carrier-associated membrane protein 1","Isoform 1 of Secretory carrier-associated membrane protein 3","Isoform 1 of Secretory carrier-associated membrane protein 4","Isoform 1 of Sentrin-specific protease 6","Isoform 1 of Seprase","Isoform 1 of Serine/threonine-protein kinase 11-interacting protein","Isoform 1 of Serine/threonine-protein kinase Nek2","Isoform 1 of Serine/threonine-protein phosphatase PGAM5, mitochondrial","Isoform 1 of Slit homolog 2 protein","Isoform 1 of Sn1-specific diacylglycerol lipase beta","Isoform 1 of Sodium/potassium-transporting ATPase subunit beta-1","Isoform 1 of Solute carrier family 12 member 2","Isoform 1 of Solute carrier family 12 member 4","Isoform 1 of Solute carrier family 12 member 9","Isoform 1 of Solute carrier family 35 member F2","Isoform 1 of Sperm-specific antigen 2","Isoform 1 of Stimulated by retinoic acid gene 6 protein homolog","Isoform 1 of Surfeit locus protein 4","Isoform 1 of Synaptic vesicle glycoprotein 2A","Isoform 1 of Synaptophysin-like protein 1","Isoform 1 of Syntaxin-10","Isoform 1 of Syntaxin-5","Isoform 1 of Syntaxin-7","Isoform 1 of TBC domain-containing protein kinase-like protein","Isoform 1 of Tenascin","Isoform 1 of Tetratricopeptide repeat protein 21B","Isoform 1 of Titin","Isoform 1 of Trans-2,3-enoyl-CoA reductase","Isoform 1 of Translocon-associated protein subunit alpha","Isoform 1 of Transmembrane emp24 domain-containing protein 4","Isoform 1 of Transmembrane protein 120A","Isoform 1 of Transmembrane protein 192","Isoform 1 of Transmembrane protein 55B","Isoform 1 of Transmembrane protein 87A","Isoform 1 of Transmembrane protein 87B","Isoform 1 of Tumor suppressor candidate 3","Isoform 1 of Ubiquinone biosynthesis protein COQ9, mitochondrial","Isoform 1 of Uncharacterized protein C4orf52","Isoform 1 of Uncharacterized protein KIAA0319-like","Isoform 1 of Uncharacterized protein KIAA1109","Isoform 1 of UPF0420 protein C16orf58","Isoform 1 of Urokinase plasminogen activator surface receptor","VAMP7, Isoform 1 of Vesicle-associated membrane protein 7","Isoform 1 of Vesicle-associated membrane protein-associated protein A","Isoform 1 of Vesicle-associated membrane protein-associated protein B/C","Isoform 1 of Vinculin","Isoform 1 of Voltage-dependent anion-selective channel protein 3","Isoform 12 of CD44 antigen","Isoform 1a of Oxysterol-binding protein-related protein 3","Isoform 1AB of Catenin delta-1","Isoform 2 of 1-acyl-sn-glycerol-3-phosphate acyltransferase gamma","Isoform 2 of 39S ribosomal protein L39, mitochondrial","Isoform 2 of 4F2 cell-surface antigen heavy chain","Isoform 2 of Acetyl-CoA carboxylase 1","Isoform 2 of Adenomatous polyposis coli protein 2","Isoform 2 of Adenosine 3'-phospho 5'-phosphosulfate transporter 1","Isoform 2 of Angiomotin-like protein 2","Isoform 2 of Annexin A2","Isoform 2 of Arginase-1","Isoform 2 of ATPase family AAA domain-containing protein 3A","Isoform 2 of ATP-binding cassette sub-family B member 6, mitochondrial","Isoform 2 of Basigin","Isoform 2 of Bcl-2-like protein 13","Isoform 2 of Calcium-transporting ATPase type 2C member 1","Isoform 2 of CAP-Gly domain-containing linker protein 1","Isoform 2 of CD97 antigen","Isoform 2 of Cell division control protein 42 homolog","Isoform 2 of Choline transporter-like protein 1","Isoform 2 of DENN domain-containing protein 5B","Isoform 2 of Desmoglein-4","Isoform 2 of Dynein heavy chain 1, axonemal","Isoform 2 of Dynein heavy chain 2, axonemal","Isoform 2 of Extended synaptotagmin-2","Isoform 2 of Fibronectin type-III domain-containing protein 3A","Isoform 2 of Filamin-A","Isoform 2 of Golgi apparatus protein 1","Isoform 2 of Heat shock protein HSP 90-alpha","Isoform 2 of Heterogeneous nuclear ribonucleoprotein A/B","Isoform 2 of HLA class II histocompatibility antigen gamma chain","Isoform 2 of Microtubule-associated protein 4","Isoform 2 of Mitochondrial import inner membrane translocase subunit TIM50","Isoform 2 of Mitochondrial inner membrane protein","Isoform 2 of Monoacylglycerol lipase ABHD12","Isoform 2 of Multidrug resistance-associated protein 1","Isoform 2 of Myosin-Ic","Isoform 2 of Myosin-VI","Isoform 2 of NADH dehydrogenase [ubiquinone] flavoprotein 3, mitochondrial","Isoform 2 of Nebulin-related-anchoring protein","Isoform 2 of Neural cell adhesion molecule 1","Isoform 2 of Neuropathy target esterase","Isoform 2 of Nucleophosmin","Isoform 2 of Oxysterol-binding protein-related protein 8","Isoform 2 of PDZ domain-containing protein 2","Isoform 2 of Plakophilin-1","Isoform 2 of Probable phospholipid-transporting ATPase IG","Isoform 2 of Protein disulfide-isomerase A6","Isoform 2 of Proto-oncogene tyrosine-protein kinase Src","Isoform 2 of Reticulon-3","Isoform 2 of Reticulon-4","Isoform 2 of Semaphorin-4B","Isoform 2 of Syntaxin-binding protein 1","Isoform 2 of Transmembrane and TPR repeat-containing protein 3","Isoform 2 of Voltage-dependent anion-selective channel protein 2","Isoform 2 of V-type proton ATPase 116 kDa subunit a isoform 1","Isoform 3 of Inositol 1,4,5-trisphosphate receptor type 1","Isoform 3 of Ribosome-binding protein 1","Isoform 3 of Sarcolemmal membrane-associated protein","Isoform 3 of Scavenger receptor class B member 1","Isoform 3 of TPR repeat-containing protein C10orf93","Isoform 3 of Tyrosine-protein kinase Fyn","Isoform 3 of Tyrosine-protein kinase-like 7","Isoform 4 of Acyl-CoA-binding domain-containing protein 5","Isoform 4 of Dedicator of cytokinesis protein 9","Isoform 4 of Inhibitor of nuclear factor kappa-B kinase-interacting protein","Isoform 4 of Mitochondrial tRNA-specific 2-thiouridylase 1","Isoform 4 of Nesprin-1","Isoform 4 of Uncharacterized protein KIAA0090","Isoform 5 of DENN domain-containing protein 1B","Isoform 5 of Interleukin enhancer-binding factor 3","Isoform 5 of Obscurin","Isoform 5 of Peptidyl-glycine alpha-amidating monooxygenase","Isoform 6 of Adenylate kinase domain-containing protein 1","Isoform 7 of Neurofascin","Isoform 9 of Caprin-2","Isoform A of Anion exchange protein 2","Isoform A of AP-1 complex subunit beta-1","Isoform A of Probable cation-transporting ATPase 13A1","Isoform A of Ras-related C3 botulinum toxin substrate 1","Isoform A of Trypsin-3","Isoform A22 of Neuropilin-2","Isoform Alpha-3A of Integrin alpha-3","Isoform Alpha-6X1X2B of Integrin alpha-6","Isoform alpha-enolase of Alpha-enolase","Isoform APP770 of Amyloid beta A4 protein (Fragment)","Isoform B of Band 4.1-like protein 3","Isoform B of Endothelin-converting enzyme 1","Isoform B of Perilipin-3","Isoform B of Phosphate carrier protein, mitochondrial","Isoform B of Plasma membrane calcium-transporting ATPase 1","Isoform B of Syntaxin-16","Isoform B of Syntaxin-3","Isoform B1 of Heterogeneous nuclear ribonucleoproteins A2/B1","Isoform Beta of Poliovirus receptor","Isoform Beta-1A of Integrin beta-1","Isoform Beta-3A of Integrin beta-3","Isoform Beta-4C of Integrin beta-4","Isoform CNPI of 2',3'-cyclic-nucleotide 3'-phosphodiesterase","Isoform Delta 6 of Calcium/calmodulin-dependent protein kinase type II subunit delta","Isoform DPI of Desmoplakin","Isoform Gnas-2 of Guanine nucleotide-binding protein G(s) subunit alpha isoforms short","Isoform LAMP-2A of Lysosome-associated membrane glycoprotein 2","Isoform Liver of ATP synthase subunit gamma, mitochondrial","Isoform Long of Endoglin","Isoform Long of Inositol 1,4,5-trisphosphate receptor type 2","Isoform Long of Sodium/potassium-transporting ATPase subunit alpha-1","Isoform Long of Thyrotropin receptor","Isoform Long of Tight junction protein ZO-1","Isoform Long of Transcription initiation factor TFIID subunit 5","Isoform Long of Tyrosine-protein kinase receptor UFO","Isoform Long of Vacuolar protein sorting-associated protein 41 homolog","Isoform Long of Very low-density lipoprotein receptor","Isoform Long of Vesicle transport through interaction with t-SNAREs homolog 1B","Isoform Long of V-type proton ATPase 116 kDa subunit a isoform 3","Isoform M2 of Pyruvate kinase isozymes M1/M2","Isoform Mitochondrial of Fumarate hydratase, mitochondrial","Isoform Non-muscle of Myosin light polypeptide 6","Isoform OA3-323 of Leukocyte surface antigen CD47","Isoform Sap-mu-0 of Proactivator polypeptide","Isoform SNAP-23a of Synaptosomal-associated protein 23","Isoform TGN51 of Trans-Golgi network integral membrane protein 2","Isoform XD of Plasma membrane calcium-transporting ATPase 4","Keratin, type I cuticular Ha1","Keratin, type I cuticular Ha8","Keratin, type I cytoskeletal 10","Keratin, type I cytoskeletal 14","Keratin, type I cytoskeletal 16","Keratin, type I cytoskeletal 17","Keratin, type I cytoskeletal 9","Keratin, type II cuticular Hb3","Keratin, type II cuticular Hb4","Keratin, type II cuticular Hb5","Keratin, type II cytoskeletal 1","keratin, type II cytoskeletal 1b","Keratin, type II cytoskeletal 2 epidermal","Keratin, type II cytoskeletal 3","keratin, type II cytoskeletal 4","Keratin, type II cytoskeletal 5","Keratin, type II cytoskeletal 6A","Keratin, type II cytoskeletal 6B","Keratin, type II cytoskeletal 6C","Kinesin-like protein KIF14","LAG1 longevity assurance homolog 2","Lamin-B receptor","Laminin subunit beta-1","Laminin subunit gamma-1","Large neutral amino acids transporter small subunit 1","Leucine-rich PPR motif-containing protein, mitochondrial","Leucine-rich repeat-containing protein 8A","Limbic system-associated membrane protein","Long-chain-fatty-acid--CoA ligase 3","Low-density lipoprotein receptor","Lysophospholipid acyltransferase 5","Lysosomal acid phosphatase","Lysosome membrane protein 2","Lysosome-associated membrane glycoprotein 1","Macrophage migration inhibitory factor","magnesium transporter protein 1","Major histocompatibility complex, class II, DP beta 1","Major vault protein","Malate dehydrogenase, mitochondrial","Mannosyl-oligosaccharide 1,2-alpha-mannosidase IB","Matrix metalloproteinase-14","MHC class I antigen","Microtubule-associated protein 1B","Mitochondrial 2-oxoglutarate/malate carrier protein","Mitochondrial chaperone BCS1","Mitochondrial glutamate carrier 1","Mitochondrial import inner membrane translocase subunit Tim23","Mitochondrial import inner membrane translocase subunit TIM44","Mitochondrial import inner membrane translocase subunit Tim9","Mitochondrial import receptor subunit TOM20 homolog","Mitochondrial import receptor subunit TOM22 homolog","Mitochondrial import receptor subunit TOM70","Mitogen-activated protein kinase scaffold protein 1","Moesin","Monocarboxylate transporter 1","Monocarboxylate transporter 4","monocarboxylate transporter 8","Myeloid-associated differentiation marker","Myosin XVIIIB","Myristoylated alanine-rich C-kinase substrate","N-acetylgalactosaminyltransferase 7","N-acetylglucosamine-6-sulfatase","NAD(P) transhydrogenase, mitochondrial","NADH dehydrogenase [ubiquinone] 1 alpha subcomplex subunit 10, mitochondrial","NADH dehydrogenase [ubiquinone] 1 alpha subcomplex subunit 4","NADH dehydrogenase [ubiquinone] 1 alpha subcomplex subunit 5","NADH dehydrogenase [ubiquinone] 1 alpha subcomplex subunit 8","NADH dehydrogenase [ubiquinone] 1 alpha subcomplex subunit 9, mitochondrial","NADH dehydrogenase [ubiquinone] 1 beta subcomplex subunit 10","NADH dehydrogenase [ubiquinone] flavoprotein 2, mitochondrial","NADH dehydrogenase [ubiquinone] iron-sulfur protein 2, mitochondrial","NADH dehydrogenase [ubiquinone] iron-sulfur protein 3, mitochondrial","NADH dehydrogenase [ubiquinone] iron-sulfur protein 5","NADH dehydrogenase [ubiquinone] iron-sulfur protein 7, mitochondrial","NADH dehydrogenase [ubiquinone] iron-sulfur protein 8, mitochondrial","NADH-cytochrome b5 reductase 1","NADH-ubiquinone oxidoreductase 75 kDa subunit","NADPH--cytochrome P450 reductase","Nascent polypeptide-associated complex subunit alpha","Neuroblast differentiation-associated protein AHNAK","Neuronal growth regulator 1","Neuronal membrane glycoprotein M6-a","neurotrimin isoform 3","Neutral amino acid transporter A","Neutral amino acid transporter B(0)","Niemann-Pick C1 protein","NKG2D ligand 2","Nodal modulator 1","novel protein similar to chondroitin sulfate GalNAcT-2","Nuclease-sensitive element-binding protein 1","Ornithine aminotransferase, mitochondrial","oxysterol-binding protein-related protein 5 isoform b","Peptidyl-prolyl cis-trans isomerase A","Peptidyl-prolyl cis-trans isomerase B","Peroxiredoxin-1","Peroxiredoxin-6","Peroxisomal membrane protein 11B","Phosphatidylinositol 3-kinase catalytic subunit type 3","Phosphatidylinositol 4-kinase type 2-alpha","Phosphoglycerate kinase 1","Phosphotidylinositol phosphatase PTPRQ","Plexin-A1","Plexin-B2","Podocalyxin-like protein 1 precursor","Poly(A) polymerase gamma","Polypeptide N-acetylgalactosaminyltransferase 2","PRA1 family protein 3","Probable phospholipid-transporting ATPase IF","Probable saccharopine dehydrogenase","Procollagen-lysine,2-oxoglutarate 5-dioxygenase 3","Profilin-1","Programmed cell death protein 6","Progressive ankylosis protein homolog","Prohibitin","Prohibitin-2","Prolactin regulatory element-binding protein","Prolactin-inducible protein","Proliferation-associated protein 2G4","Prolow-density lipoprotein receptor-related protein 1","Prostaglandin F2 receptor negative regulator","proteasome-associated protein ECM29 homolog","protein ALO17 isoform 1","Protein disulfide-isomerase","Protein disulfide-isomerase A3","Protein disulfide-isomerase A4","Protein ERGIC-53","Protein FAM162A","Protein FAM3C","Protein kinase C and casein kinase substrate in neurons 3, isoform CRA_b","Protein NOXP20","Protein S100-A10","Protein S100-A11","Protein S100-A6","Protein S100-A7","Protein S100-A8","Protein S100-A9","Proteolipid protein 2","Protocadherin beta-8","Proto-oncogene tyrosine-protein kinase Yes","Putative chloride channel protein 7","Putative uncharacterized protein ALB","Putative uncharacterized protein ATP5J2","Putative uncharacterized protein CASP12","Putative uncharacterized protein SDHA","pyruvate dehydrogenase E1 alpha 1 isoform 2 precursor","Radixin, isoform CRA_a","Ras GTPase-activating-like protein IQGAP1","Ras-related protein Rab-10","Ras-related protein Rab-11B","Ras-related protein Rab-14","Ras-related protein Rab-21","Ras-related protein Rab-2A","ras-related protein Rab-31","Ras-related protein Rab-35","Ras-related protein Rab-5C","Ras-related protein Rab-7a","Ras-related protein Ral-A","Ras-related protein Rap-1b","Ras-related protein Rap-2b","Ras-related protein Rap-2c","Ras-related protein R-Ras","Regulator of G-protein signaling 19","Renin receptor","Replication protein A 70 kDa DNA-binding subunit","Rho GDP-dissociation inhibitor 1","RhoA activator C11orf59","Rho-related GTP-binding protein RhoB","Ribosomal protein L14 variant","Ribosomal protein S27","Sec1 family domain-containing protein 1","Secretory carrier-associated membrane protein 2","Semaphorin-7A","Serotransferrin","Serpin H1","Sideroflexin-1","sideroflexin-3","Signal peptide peptidase-like 2A","signal-regulatory protein alpha precursor","similar to major histocompatibility complex, class II, DR beta 5 isoform 2","Similar to Protocadherin alpha-1 precursor","similar to ubiquitin protein ligase E3 component n-recognin 5 isoform 1","Sin3 histone deacetylase corepressor complex component SDS3","Single-stranded DNA-binding protein, mitochondrial","Sodium/potassium-transporting ATPase subunit beta-3","Solute carrier family 2, facilitated glucose transporter member 1","Solute carrier family 4 sodium bicarbonate cotransporter member 7","Sortilin","Sorting and assembly machinery component 50 homolog","Staphylococcal nuclease domain-containing protein 1","Sterol O-acyltransferase 1","Stomatin-like protein 2","Stress-70 protein, mitochondrial","Succinate dehydrogenase [ubiquinone] iron-sulfur subunit, mitochondrial","Succinyl-CoA ligase [GDP-forming] subunit beta, mitochondrial","Sulfate transporter","Sulfide:quinone oxidoreductase, mitochondrial","Superoxide dismutase [Mn], mitochondrial","Synaptic vesicle membrane protein VAT-1 homolog","Synaptogyrin-2","Synaptonemal complex protein 1","Syntaxin-12","Syntaxin-6","Talin-1","TBC1 domain family member 5","T-complex protein 1 subunit beta","T-complex protein 1 subunit delta","T-complex protein 1 subunit epsilon","T-complex protein 1 subunit eta","T-complex protein 1 subunit gamma isoform b","T-complex protein 1 subunit zeta","Thioredoxin","Thioredoxin-related transmembrane protein 4","Thymosin beta-4-like protein 3","TMEM181 protein (Fragment)","Transducin beta-like protein 2","Transferrin receptor protein 1","Transforming protein RhoA","Transgelin-2","Transitional endoplasmic reticulum ATPase","Translation initiation factor eIF-2B subunit epsilon","Translational activator GCN1","Transmembrane 9 superfamily member 2","Transmembrane 9 superfamily member 4","Transmembrane emp24 domain-containing protein 10","Transmembrane emp24 domain-containing protein 2","Transmembrane emp24 domain-containing protein 7","Transmembrane emp24 domain-containing protein 9","Transmembrane protein 165","Transmembrane protein 2","Transmembrane protein 43","Transmembrane protein C2orf18","Tricarboxylate transport protein, mitochondrial","Trifunctional enzyme subunit alpha, mitochondrial","Trifunctional enzyme subunit beta, mitochondrial","triosephosphate isomerase isoform 2","Tripeptidyl-peptidase 2","tropomyosin alpha-3 chain isoform 1","Trypsin-1","Tu translation elongation factor, mitochondrial precursor","Tubulin beta-2A chain","Tubulin beta-2C chain","tumor protein D54 isoform a","Tyrosine-protein kinase SgK269","Tyrosine-protein phosphatase non-receptor type 1","Ubiquinone biosynthesis monooxygenase COQ6","Uncharacterized glycosyltransferase AGO61","Uncharacterized protein KIAA1467","UPF0539 protein C7orf59","Vacuolar protein sorting-associated protein 45","Valyl-tRNA synthetase","Vesicle transport through interaction with t-SNAREs homolog 1A","Vesicle-fusing ATPase","Vesicular integral-membrane protein VIP36","Vimentin","Voltage-dependent anion-selective channel protein 1","V-type proton ATPase 116 kDa subunit a isoform 2","V-type proton ATPase catalytic subunit A","V-type proton ATPase subunit B, brain isoform","V-type proton ATPase subunit d 1","V-type proton ATPase subunit S1","Zinc finger protein 729","Zinc finger protein-like 1","Zinc transporter 7","Zinc transporter ZIP10","ZNF507 protein"],[12.2,11.4,16.2,36.2,5.8,12.23809524,32.11111111,5.923076923,30.6,23.4,1,1,1,1.444444444,1.888888889,1,0.619047619,1,1,0.5862068966,1,1.64,1,1,1,0.5555555556,1,1.8,3.4,1,1,1,0.4482758621,0.5555555556,3.4,7.4,1,1,1,1,1.8,0.3846153846,1,1.8,4.2,0.6923076923,1,1,1.8,0.8222222222,1.484848485,0.6923076923,0.2380952381,1,0.5555555556,0.3846153846,1.615384615,1,2.6,1,1,1,1,1,0.5555555556,0.1020408163,1,2.6,1.689655172,4.2,1,1.8,1,1,1,1,0.52,2.6,1.177777778,1,0.5555555556,1,1,1,1,1,2.6,1.8,2.6,1,0.2380952381,1,0.1827956989,1,0.1111111111,0.1515151515,0.603960396,1,0.8620689655,1.244897959,2.6,0.8095238095,1,1,1.8,1,1,0.4285714286,1,1,5,0.296,1,1.8,1.8,1,0.2941176471,1,1,1,0.2941176471,3.4,1,0.52,1,0.3846153846,0.5555555556,1,0.2380952381,3.4,1,1,3.4,2.6,1,0.2,2.621621622,0.2101910828,1,1,1,1,0.3103448276,0.1007751938,0.1238095238,1,1,4.2,1.8,3.4,5,1.8,3.4,1.8,1.8,1,1,1,1,1,3.4,1,1.307692308,1.444444444,1,1,1.8,1,1,1,0.36,1,1.923076923,6.6,10.6,1,0.7647058824,1,0.5555555556,0.3846153846,1.8,1,0.52,1,1,1,1,1,0.2195121951,1,2.333333333,0.5555555556,0.2,4.529411765,1,0.3846153846,1.8,1,0.3846153846,1,0.02578796562,0.03614457831,1,0.5555555556,1,0.5555555556,1,1.137931034,6.538461538,1,1.444444444,0.5555555556,1,5.8,0.5555555556,1,1,4.2,2.6,1,1,2.6,1,1,1,0.2941176471,1,0.1219512195,1,1,1,0.2380952381,2.6,1,0.5555555556,0.5555555556,0.3846153846,0.5555555556,0.4639175258,0.619047619,1,1,1,1,1,1,1.606060606,1,1,1,0.08196721311000001,0.3846153846,2.6,1,0.2941176471,1,1,1,1,3.4,1,0.4285714286,0.5555555556,1,1,1,1,1,1,2.6,1.8,1,1,1,1,1,0.68,0.2941176471,1,1.075471698,0.2941176471,0.5266272189,0.4285714286,0.2941176471,0.5555555556,1.8,0.7647058824,1,0.2941176471,1,0.5294117647,0.5555555556,0.5555555556,1,1,1,1,1.8,0.5675675676,1,1,0.1724137931,1,2.6,1,1,1,2.6,2.6,1,1,0.3846153846,1,1.8,1,1.8,1,1,1,1,1,2.6,0.2380952381,0.3846153846,1.64,1,1,1,2.777777778,1,3.823529412,1,0.2941176471,1,1,1,0.2380952381,1,1,1,0.1724137931,1,1,0.3846153846,1,1.235294118,1,0.2380952381,1,0.5555555556,1,1,0.5675675676,5.888888889,0.5555555556,0.1351351351,1,1,1,1,1.8,2.6,0.02200488998,0.5555555556,1,1.444444444,0.3846153846,1,0.5555555556,1,0.4285714286,0.3846153846,2.6,1,1,1.8,1,0.2,1,1,1,1,1,1,1,1,1,0.5555555556,1,1.8,1,0.3846153846,1.8,5.8,2.6,1,1,1,0.6226415094,3.461538462,2.081081081,1.8,1,1.888888889,1,5,1,1.444444444,1,0.2941176471,2.6,1,0.5121951219999999,1,1,1,0.2,0.52,0.737704918,3.222222222,1,1.96,1.8,0.2380952381,1,0.3846153846,1.8,1,1,1,1,1.8,0.3469387755,1.8,1,1,2.6,0.5555555556,1,1,0.5555555556,9,1.307692308,2.6,1,1.8,2.818181818,1,1,1,1,0.7575757576,1,1,1,1,5.307692308,1,1,0.8620689655,0.7333333332999999,1,0.2,1,3.4,1,1,1,0.2380952381,1,0.3846153846,0.5555555556,1,1,0.06172839506,0.5555555556,1,1,1,1,0.06849315068,1,0.3846153846,1,1,1,1.571428571,1,1,1,1,1,0.5555555556,1,0.2941176471,1,3.4,0.05617977528,0.3846153846,1,1,0.3513513514,0.1688311688,0.3846153846,1,1,1,1,2.6,1,1,1,0.3846153846,1,1,0.5555555556,1,1,1,1,0.2941176471,0.3846153846,1,1,1,1,1,4.2,1,0.8620689655,1.8,0.04424778761,1,1,0.5555555556,0.2941176471,1,1,6.6,0.9024390244,1,0.5555555556,0.6705882353,0.2,0.2,1,1,0.7647058824,0.2941176471,0.2195121951,1.888888889,1,1,0.5406698565,1,1,1,1,1,0.2,1.8,1.307692308,4.2,1,1,1,0.6923076923,4.2,0.2941176471,1,0.52,1,0.6032876712,1.407643312,5.615384615,4.2,0.7624831309,0.5862068966,1.615384615,0.2941176471,1.056440601,1,1.077727952,1,1,1.198895028,1.8,1.8,1.393442623,1,1,1,1,1,1,0.3846153846,1,4.2,1,1,1,2.6,0.4754098361,1,1,1,1,0.5555555556,0.6923076923,0.5555555556,1.8,2.6,1.444444444,1,1,1,1.8,1,1,1,1,1,3.4,0.0495049505,2.846153846,0.4482758621,2.6,5,1,1,1,0.3846153846,1,1,1,1.8,1,1,1.8,1.8,0.1020408163,1,1,1,1.8,0.4285714286,0.3103448276,0.4285714286,1,0.2195121951,1,1.444444444,1,0.5555555556,4.2,0.2432432432,0.7647058824,1,1,1,1,1,2.6,1.8,1,1,1,1,1,0.1688311688,1,1,1,1.8,1,1,1,1,1,1,2.6,1,1,2.846153846,0.52,1,1.8,1,0.2941176471,0.2941176471,0.3846153846,1,3.4,2.6,0.2380952381,1,3.4,4.2,1,1,1,1,1,0.5555555556,0.6923076923,0.3846153846,4.2,1,6.6,2.6,4.2,1,1.503644798,2.6,1,1,1,1,1.8,2.6,1,2.6,1,1,4.2,6.6,3.4,1.8,3.4,1,5,1.8,1,1,1,5,1,1,3.4,1,1,0.5294117647,0.2941176471,1,1.8,1,1,0.2941176471,1,1,1,1,1,1.8,1.413793103,1,2.538461538,1,1,1,2.6,3.4,1,1,1,0.2380952381,1,1,3.4,0.5555555556,1.137931034,0.7647058824,1,1,5,2.6,1,3.4,1.8,1.8,1,1,1,1.8,1,0.2996632997,1,1.8,0.5294117647,1.888888889,0.2727272727,0.5068493151,0.3939393939,3.4,1,1,0.6923076923,0.52,1,1,1.923076923,0.5294117647,0.4482758621,0.1111111111,1,1,1.8,1.8,1,1,1,1.8,0.5294117647,1,1,1,1,1,1.8,1,1.8,0.09433962264,1,0.2,0.4352941176,1.8,1,1.8,1.888888889,0.619047619,1,1,1,1,1],[0.84,1.110091743,1,2.6,4.555555556,5,2.038961039,5,14.6,5.235294118,1,1,1,0.2380952381,0.3846153846,2.6,1.380952381,3.4,1.8,2.44,1.8,1.489795918,1.8,1,1,1.8,1,1,1,1,1,1,1,0.3846153846,1,20.2,0.5555555556,1.8,1,1,1,1,1,3.4,0.68,0.6101083032,1,2.6,1,0.4385964912,0.5061728395,1,4.047619048,1,5,1.8,1,1,1.324324324,1.8,1.8,1,1,1,1,5.8,16.2,1,0.8810408922,5,1,1.8,1,1,0.5555555556,1,1,1.8,6.6,1,1.8,1,1.8,4.555555556,1,1.8,1.571428571,1.8,1,1,1,1,0.3846153846,1,2.6,1,0.2380952381,1,1.8,2.6,3.4,1,3.4,0.1384615385,1,0.5555555556,1,2.6,1,5.8,1.8,1.8,1,3.4,1,1,1,1,5.8,1.8,1.307692308,2.6,1,0.4594594595,1.8,1,0.3846153846,0.3846153846,1,1,1,3.4,2.230769231,1.235294118,1.8,1,2.6,1.819512195,1,2.6,3.4,1,1,2.76,2.081081081,1,1,1.888888889,1,1.8,1.8,1.8,1,1,1,1.444444444,1,1,1,1,1,1,0.3103448276,0.1020408163,1,1,2.6,1.8,1,1,0.5555555556,1,1.380952381,1.8,4.384615385,1,0.3846153846,1,2.6,1.615384615,1,1.8,6.6,3.4,1,1,1,1,10.6,1,2.28,1,1,5,1,1.8,2.6,1,1,1,5.307692308,2.333333333,1,1,1,2.6,1,1,1.8,1,1,1,1,1,1,1,1,3.4,1,3.4,1.444444444,1,1.8,2.411764706,1,1.8,1,3.4,1,1,1,1,1,1,1,1,1,1,1.905660377,1,1,1,1,1,0.1515151515,1,5.727272727,3.4,1,1,0.3846153846,1,2.6,1,1,1,1,1,1,1,1,3.4,1.8,1,1.8,2.6,1,1,2.6,1.307692308,2.6,5,1,1,1,1.8,1,1.444444444,2.6,1.235294118,1.615384615,3.4,1.470588235,1.8,1,1,1,1,1,0.7419354839,1.8,1,1,1,1,1,1,2.411764706,1,1,1,1,0.2941176471,1,1,1,1,1.8,1,1,4.2,3.4,1,0.5555555556,1,1,1,1,1,1.8,1,1,3.4,1.888888889,4.384615385,1,1,1,1.561403509,1,2.795918367,1,1,1,1,1,1,3.4,1,1,1,0.1219512195,0.5555555556,1,1,1,1,0.5294117647,1,1,1,1,2.6,2.333333333,1,1,1.8,7.4,0.3846153846,1,1,1.8,0.3709198813,1,0.2941176471,0.2941176471,3.4,1,1,1,1,1.8,2.6,1,0.5555555556,1.888888889,1,0.1780821918,1,1.235294118,1,4.2,1,1,1,1,2.6,1,1,1,2.6,1,2.6,2.6,1,1,0.3846153846,0.5555555556,0.8787878788,1,0.5555555556,1.8,1,1,1,2.333333333,1,2.6,1,2.6,1.8,1.8,1,1,1,1,1,1,1.8,1,1,1.444444444,1.470588235,1,1,0.36,0.3846153846,1,1,1,1,1,1,1,1,1,1,1,1,1,0.5555555556,1.8,2.44,4.2,1,1,4.529411765,1,2.6,1,1,4.034482759,0.6923076923,2.6,1,0.3846153846,0.7763975155,1,3.4,1.8,2.714285714,1,1,0.6923076923,1,1,1,0.619047619,1,1,4.764705882,0.5555555556,1,0.1724137931,1.235294118,2.846153846,1.8,1,2.333333333,2.538461538,2.073170732,1,1,1,1,1,0.5409836066,1,1,1,1,1,1,0.5555555556,1,0.5555555556,1,3.4,1,1,1,2.170731707,1.8,1,1,1,1,1,1,1,1,3.4,3.4,0.3846153846,2.6,1,1,2.6,2.6,1,7.4,1,0.5555555556,1,1,1,1.8,6.6,9,1.177777778,1,2.945945946,2.6,1,1,1,3.352941176,1,1,1,1,1,1.19047619,0.2941176471,1,1,1,0.3513513514,1,5,1.8,1,1,2.015873016,1,1,1,1,1,0.5555555556,1,1,4.52,5,1,1,1,2.6,0.5555555556,2.647058824,1,1,0.6763080923,0.4838709677,0.0693877551,0.2941176471,0.2094820922,1,0.2941176471,1,0.5431956558,2.6,0.9813809154000001,0.3846153846,0.2,0.3652392947,0.2,0.2,0.08300395257,3.153846154,1,1,1,1,1,3.4,1,1,1,4.2,1,1,1.827586207,1,1,1,1,1,7.4,1,1,1,1,1,1,1,1,1,1,1.8,4.2,1.8,1,1.195121951,1,1,1,1,1,0.3846153846,1,1,1.941176471,2.230769231,1,1.8,1,1,1.8,1,1.888888889,4.2,1,1,2.6,1,6.6,1,1,1.8,1,1,1,1.8,2.931034483,1,1,0.5555555556,1,1.8,2.6,1,6.6,1.8,5,1,1,1.8,1,3.4,0.2786885246,1,1,0.7647058824,0.5555555556,0.6923076923,1,1,1,1,5,1,1,1.19047619,1,1,1,1,1,0.6923076923,0.5555555556,1,1.8,3.4,5.8,1,0.5555555556,1,1,1,1,1,0.2380952381,1.8,1,0.5555555556,1,1,1,1,0.2481203008,1.8,2.821474774,3.4,1.8,1.8,1,2.6,1,0.84,1,5.8,1,3.4,2.333333333,16.2,2.6,1.8,1,1,1.8,1,1,1,1.444444444,1,1,1.8,1,1,1,0.2941176471,0.3846153846,1,1,0.5555555556,1,1,1,1,0.3846153846,0.1952662722,4.2,2.333333333,1.615384615,1.8,1,0.5555555556,1,1,1.648648649,25,2.6,2.6,1,1,3.4,0.3846153846,1,1.8,0.3846153846,1,3.4,1,1,2.6,1.8,1,1.8,1,2.6,1,1,1,1,3.731707317,1,1.8,3.117647059,0.7837837838,1,1.8,1,3.4,1,1,1,1,1,1,1,1,1,3.4,1,1,1,0.7647058824,1,1,1,1,1,1,1,1,0.5555555556,1,1.8,1,1,1,1,0.6279069767,1.553846154,1,1,1,1.761904762,1,0.3846153846,1,1,1,0.3846153846],[1.622222222,1.727272727,2.777777778,1.941176471,3.4,1.524590164,30.45454545,0.3442622951,4.169811321,0.7575757576,1,1,1,0.6923076923,1,1,1.8,1,0.3846153846,0.1698113208,1,3.731707317,1,1,1,1,1,1,1,1,1,1.8,1,1,1,0.3846153846,1,1.8,1,1,1,1,1,1.888888889,1,1,1,3.4,1,2.333333333,3.823529412,1,0.5151515152,1,2.76,1,1.8,2.6,0.5294117647,1,5,0.6923076923,1,1,1,1,1.307692308,1,0.6694214876,2.6,1,2.6,1,1,2.6,0.3846153846,3.4,1.8,0.3513513514,1.8,1,1,1,1.8,1,1.8,1.8,2.6,1,4.2,1,1,0.7735849057,1,2.6,1.235294118,0.8651685393,1,1,1.615384615,1,1.8,9.888888889,2.230769231,1,1,1,1.380952381,1,1,1,11.4,1,3.4,1,1,1,1,1,3.4,0.2195121951,1,1,2.142857143,1,1,1.307692308,1,1,1.8,1,1.8,1,1,1,0.06493506494,0.6363636364,0.6674364896,1,0.2941176471,1,0.3846153846,0.36,2.6,6.6,1,2.6,2.6,1,1,4.2,5,0.5555555556,1,1,1,1,1,1.8,1,1,1,1,3.476190476,1,1,1,1,0.2941176471,0.2941176471,3.666666667,1,9.800000000000001,5.8,2.6,1.8,1,1,0.5555555556,0.6923076923,3.4,1,0.6363636364,2.333333333,1,1,1,1,27.4,1,0.8095238095,0.3846153846,1,3.153846154,1,5,0.3846153846,1,2.6,2.6,3.344827586,3.620689655,1,1,0.3846153846,0.3170731707,1,1,1.8,1,2.6,1,1,1.8,1,1,1.8,4.2,1,0.2195121951,0.6923076923,0.2207792208,0.3103448276,1,1,1,5,3.24,1,1,1,1,1,1,3.4,1,1,0.6923076923,0.8222222222,1,1,1,1,1,8.199999999999999,1,2.170731707,1,1,1,1.8,1,1,1,2.333333333,9,1,0.8532110092,1,1,2.6,1,1,1,3.4,1,1,1,1,1,2.6,1,2.6,1,1.8,1,2.6,1.8,1,0.9024390244,0.4285714286,0.8032786884999999,0.7735849057,1,1,2.6,1,3.4,1,1,0.2380952381,1,1,1,1,1,1,1,1,1,1,1,1.8,1,1,1,1,1,1,1,1.470588235,1,1,1.8,3.4,1,1,1.8,0.3846153846,1,2.6,1,1,0.5555555556,4.2,0.5555555556,1,1,0.3684210526,1,1.547945205,1,1,1,1,1,1,1.8,1,1,0.1351351351,3.4,1.307692308,1.8,1,1,1,1.088888889,1,1,1,1,1,0.4285714286,1,1,1.8,1,1,1,1.8,4.2,1.943683409,2.6,1.8,3.823529412,1.8,1,1,1.8,1.444444444,1,5,1,1,0.2941176471,0.5555555556,0.6923076923,1,0.1219512195,1,3.222222222,1,1,1,3.4,1,1,1,1,1,2.6,1,2.714285714,12.2,1,1,1,5,1,1,2.6,1.8,1,1,0.2380952381,1,1.470588235,1,1.444444444,0.5555555556,0.68,2.538461538,2.6,1,1,1,1.8,0.4594594595,3.4,1,1.380952381,2.6,1,0.2941176471,0.3846153846,1,1.8,1,1,1.8,1,2.6,1,1,1,1,1.444444444,1,1,1,1.761904762,1.941176471,6.6,1.615384615,2.6,0.9298245614,1,1.888888889,1,1,0.8759689922,0.2941176471,0.3846153846,1,1,0.4482758621,5,1,1,2.658536585,1,1,0.6923076923,1.615384615,1,1,1.307692308,0.5555555556,1.8,0.3663366337,1,1,0.8095238095,2.6,0.5555555556,1,1,0.5555555556,1.8,4.2,1.8,1,1,1,1,3.769230769,0.3939393939,1,1,1,0.5555555556,1,0.6923076923,2.6,1,1,0.6923076923,1,1,1,3.222222222,1,0.5555555556,1,1,1,3.4,1,1,1.8,2.333333333,1,1,0.3846153846,1,1,1,0.3846153846,1,0.5789473684000001,1,2.6,1,1,1,5,0.4146341463,1.307692308,0.4248366013,1,0.6226415094,1,1,0.649122807,1,3.4,1,1,2.6,1,1,0.3984962406,1.307692308,1.444444444,0.5555555556,1,2.92,1.941176471,0.2,2.6,1,1,1.768683274,0.5555555556,2.6,1,0.6923076923,0.3846153846,2.939393939,1,0.6923076923,3.068965517,1,1,1.444444444,2.6,1,2.6,1,0.0289017341,2.6,1.00612323,3.480243161,11.66666667,39.4,1.232727273,0.08771929824999999,0.1011235955,0.2,1.148424737,1.444444444,1.43697479,1.8,1,3.054474708,5,1,6.295238095,0.5555555556,1,1,1,1,1.8,1,1.444444444,1,1,2.6,1,0.6923076923,6.647058824,1,1,1,1.8,1,0.3846153846,1,0.7575757576,0.2280701754,1,1,1,1,1,2.6,1,1,1.8,1,1.8,4.04,3.4,1,1,0.2941176471,0.3846153846,1,1,1,2.6,1,1.8,1.8,1,1.8,1,1,2.6,5,1,1,1.8,1,0.5555555556,2.6,1,1.444444444,1,1.8,1,1.8,1.061538462,1.888888889,1.8,1,0.5555555556,1,1,0.5555555556,2.777777778,1.8,1,1,1,1,1,0.3513513514,3.666666667,1.307692308,0.5555555556,1.32,1,1,1,1,1,1,1.615384615,1,1,13.8,7.4,1,4.2,1,1,1.941176471,1,1,5,1.888888889,1,1,1,1,1,1,1,1,1,1.8,1,4.111111111,1,1,5,1,2.230769231,1,0.5355041193,1.8,1,1,0.6923076923,0.2941176471,2.6,2.76,1,9.800000000000001,2.6,1.444444444,1.923076923,7.666666667,1.888888889,1,1.444444444,1.8,2.6,1,1,1,2.6,1.444444444,1,1,1,1,1,1.235294118,1,1,3.4,1.8,1,1,1,1,1,3.117647059,1,1.590163934,1.164383562,1,1.8,1,0.5555555556,1,1.8,3.666666667,2.6,1,1,1,1,0.5555555556,1,2.6,1.307692308,2.6,0.5555555556,1,1,1,1,1.8,1,1,5,1,1,1,1,1.156479218,1,1,5,0.1698113208,1,2.333333333,2.6,1.888888889,1,1,2.6,1,3.4,1,1,1,1,1,0.5555555556,1,1,1,1,0.5555555556,1,1.705882353,1,1,1,1,1,1,0.2941176471,1,1.8,5.8,1,0.2,12.2,1,0.5555555556,1,0.2941176471,0.3846153846,0.3846153846,1,1,1,1]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Protein Name<\/th>\n      <th>Change in siASM<\/th>\n      <th>Change in Desi<\/th>\n      <th>Change in Mangosteen<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"ftp","paging":true,"columnDefs":[{"className":"dt-right","targets":[2,3,4]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"name","targets":1},{"name":"change","targets":2},{"name":"change_desi","targets":3},{"name":"change_mango","targets":4}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>


</section>

<a onclick="window.scrollTo(0, 0); return false;" role="button" id="quarto-back-to-top"><i class="bi bi-arrow-up"></i> Back to top</a></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
    const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
    const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
    let newTheme = '';
    if(darkModeDefault) {
      newTheme = isAlternate ? baseTheme : alternateTheme;
    } else {
      newTheme = isAlternate ? alternateTheme : baseTheme;
    }
    const changeGiscusTheme = () => {
      // From: https://github.com/giscus/giscus/issues/336
      const sendMessage = (message) => {
        const iframe = document.querySelector('iframe.giscus-frame');
        if (!iframe) return;
        iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
      }
      sendMessage({
        setConfig: {
          theme: newTheme
        }
      });
    }
    const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
    if (isGiscussLoaded) {
      changeGiscusTheme();
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  const darkModeDefault = true;
  let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
    toggleGiscusIfUsed(toAlternate, darkModeDefault);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp("https:\/\/github\.com\/clfee\/website");
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
          // target, if specified
          link.setAttribute("target", "_blank");
          if (link.getAttribute("rel") === null) {
            link.setAttribute("rel", "noopener");
          }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      &nbsp;
    </div>   
    <div class="nav-footer-center">
<p>© 2024 · Made with r-project and <a href="https://quarto.org/"><img src="https://quarto.org/quarto.png" class="img-fluid" alt="Quarto" width="65"></a></p>
</div>
    <div class="nav-footer-right">
      <ul class="footer-items list-unstyled">
    <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/clfee/website/">
      <i class="bi bi-github" role="img">
</i> 
    </a>
  </li>  
</ul>
    </div>
  </div>
</footer>




<script src="../site_libs/quarto-html/zenscroll-min.js"></script>
</body></html>